From f4bf4be5bfb731b218b247f89c9ff8bd2c2833d9 Mon Sep 17 00:00:00 2001
From: Manoj Srivastava <srivasta@golden-gryphon.com>
Date: Sun, 27 Apr 2014 22:02:31 -0700
Subject: [PATCH 1/4] debcherry fixup patch

3033685 [fix-man-page-typo]: Fix man page typos.
	 - no changes against upstream or conflicts
f749bdf [master[: First step towards moving to a new build system.
	 - no changes against upstream or conflicts
cfb9a7d [topic--man-fixes]: Fix man page comment schema
	 - extra changes or conflicts
4e1b636 Added the debian submodule.
	 - extra changes or conflicts
20c4ead Fix the buggy invocation of split
	 - extra changes or conflicts
48d7579 Incorporate previous changes to dist
	 - extra changes or conflicts
371472d Initial import of upstream branch
	 - extra changes or conflicts
---
 ChangeLog               |  38 ++++----
 MANIFEST                |   2 +-
 bin/manicheck.man       |  30 +++---
 bin/manilist.man        |  48 +++++-----
 bin/packinit.SH         |   1 +
 bin/packinit.man        |  56 +++++------
 bin/perload             |   8 +-
 dist.man                |  70 +++++++-------
 jmake/bindex.SH         |   2 +-
 jmake/files/Jmake.rules |  10 +-
 jmake/fixcpp.SH         |   2 +-
 jmake/jmake.SH          |   2 +-
 jmake/jmake.man         |  92 +++++++++---------
 jmake/jmkmf.man         |  34 +++----
 kit/kitpost.man         |  34 +++----
 kit/kitsend.man         |  34 +++----
 kit/makeSH.man          |  30 +++---
 kit/makedist.man        |  54 +++++------
 kit/manifake.man        |  34 +++----
 mcon/Jmakefile          |   2 +-
 mcon/Makefile.SH        |   2 +-
 mcon/NOTES              |   2 +-
 mcon/U/Configdir.U      |   2 +-
 mcon/U/Finish.U         |   2 +-
 mcon/U/Guess.U          |   2 +-
 mcon/U/Head.U           |   2 +-
 mcon/U/Inhdr.U          |   2 +-
 mcon/U/Inlibc.U         |   2 +-
 mcon/U/Setvar.U         |   2 +-
 mcon/U/Warn_v7EXT.U     |   2 +-
 mcon/U/Warn_v7ND.U      |   2 +-
 mcon/U/abortsig.U       |   2 +-
 mcon/U/afs.U            |   2 +-
 mcon/U/bin.U            |   2 +-
 mcon/U/ccflags.U        |   2 +-
 mcon/U/d_bsdjmp.U       |   2 +-
 mcon/U/d_eofpipe.U      |   2 +-
 mcon/U/d_gethname.U     |   2 +-
 mcon/U/d_keepsig.U      |   2 +-
 mcon/U/d_newsadm.U      |   2 +-
 mcon/U/d_normsig.U      |   2 +-
 mcon/U/d_setpgrp.U      |   2 +-
 mcon/U/d_speedopt.U     |   2 +-
 mcon/U/d_tzmin.U        |   2 +-
 mcon/U/d_vfork.U        |   4 +-
 mcon/U/i_termio.U       |   2 +-
 mcon/U/myhostname.U     |   2 +-
 mcon/U/nametype.U       |   2 +-
 mcon/U/nis.U            |  12 +++
 mcon/U/registers.U      |   2 +-
 mcon/U/sbrksmart.U      |   2 +-
 mcon/U/shm_for.U        |   2 +-
 mcon/U/sysman.U         |   2 +-
 mcon/U/uidtype.U        |   2 +-
 mcon/U/voidflags.U      |   2 +-
 mcon/U/yacc.U           |   2 +-
 mcon/man/mconfig.SH     | 248 ++++++++++++++++++++++++------------------------
 mcon/man/mlint.SH       | 102 ++++++++++----------
 mcon/man/mxref.SH       |  58 +++++------
 pat/pat.man             | 102 ++++++++++----------
 60 files changed, 595 insertions(+), 582 deletions(-)

diff --git a/ChangeLog b/ChangeLog
index 3dce502..0726af4 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -61,7 +61,7 @@ Fri Feb 28 17:48:41 MET 1997   Raphael Manfredi <Raphael_Manfredi@grenoble.hp.co
 
 	Added the srcdir support to "configure" now that we have src.U.
 
-	Documented the runnning environment and the src.U unit in
+	Documented the running environment and the src.U unit in
 	the metaconfig man page. The $_a and $_o are also mentionned.
 	All the generated scripts must now start with a "$startsh"
 	to ensure proper shell execution.
@@ -90,7 +90,7 @@ Fri Feb 28 17:48:41 MET 1997   Raphael Manfredi <Raphael_Manfredi@grenoble.hp.co
 	mcon/U/patchlevel.U, mcon/U/MailAuthor.U, mcon/U/Chk_MANI.U,
 	mcon/U/Config_h.U: Added support for src.U.
 
-	* mcon/U/Configdir.U: Have README explicitely mention the package
+	* mcon/U/Configdir.U: Have README explicitly mention the package
 	name.
 
 	* mcon/U/make.U, mcon/U/d_strtoul.U, mcon/U/Unix.U,
@@ -123,7 +123,7 @@ Fri Feb 28 17:48:41 MET 1997   Raphael Manfredi <Raphael_Manfredi@grenoble.hp.co
 
 	* mcon/U/Whoa.U: Whoa script now starts with leading "startsh".
 
-	* mcon/U/afs.U: Can now explicitely tell Configure whether AFS is
+	* mcon/U/afs.U: Can now explicitly tell Configure whether AFS is
 	running.
 
 	* mcon/U/archlib.U: Skip existence checks for archlib.
@@ -265,7 +265,7 @@ Fri Feb 28 17:48:41 MET 1997   Raphael Manfredi <Raphael_Manfredi@grenoble.hp.co
 	* mcon/configure: Added the srcdir support now that we have src.U. 
 	Random cleanup for nicer help message.
 
-	* mcon/man/mconfig.SH: Documents the runnning environment and the
+	* mcon/man/mconfig.SH: Documents the running environment and the
 	src.U unit.  Added warnings for $_a and $_o, as well as "startsh".
 
 	* mcon/man/mlint.SH: New "create" and "empty" lint directives. 
@@ -969,7 +969,7 @@ Wed Jan 11 17:03:22 MET 1995   Raphael Manfredi <ram@acri.fr>
 	Andreas Koenig <k@franz.ww.tu-berlin.de> on the perl5-porters
 	mailing list. Thanks!
 
-	Now sets sbrksmart to undef explicitely when lacking sbrk().
+	Now sets sbrksmart to undef explicitly when lacking sbrk().
 	Forgot a cast when using return value from sbrk(). Both fixes
 	reported by Xavier Le Vourch <xavierl@eiffel.com>.
 
@@ -1038,7 +1038,7 @@ Wed Jan 11 17:03:22 MET 1995   Raphael Manfredi <ram@acri.fr>
 	* mcon/U/orderlib.U: Allows hint files to specify their own value for
 	'ranlib'.
 
-	* mcon/U/sbrksmart.U: Now sets sbrksmart to undef explicitely when
+	* mcon/U/sbrksmart.U: Now sets sbrksmart to undef explicitly when
 	lacking sbrk().  Forgot a cast when using return value from sbrk().
 
 	* mcon/U/spitshell.U: Use 'test -f' instead of 'test -r' for
@@ -1083,7 +1083,7 @@ Mon Oct 31 10:57:05 MET 1994   Raphael Manfredi <ram@acri.fr>
 	in their ?MAKE: line due to the way metalint performs its visible
 	symbol lookup (by following dependencies recursively and implicitely
 	placing Init at the top). Since Options is no longer a dependency
-	of Init, Myread.U and Csym.U now explicitely mention Options in
+	of Init, Myread.U and Csym.U now explicitly mention Options in
 	their dependency line.
 
 	Also, the leading comment in Chk_MANI.U now explains how this
@@ -1302,12 +1302,12 @@ Sat Oct 29 18:16:03 MET 1994   Raphael Manfredi <ram@acri.fr>
 	HOME to the ?T: line since metalint now checks ${HOME}.
 
 	* mcon/U/Guess.U: Added ?F: line for metalint file checking.  Call
-	./xenix explicitely instead of relying on PATH.
+	./xenix explicitly instead of relying on PATH.
 
 	* mcon/U/Head.U: Make sure ENV is unset before calling /bin/ksh.
 
 	* mcon/U/Setvar.U, mcon/U/Inhdr.U, mcon/U/Inlibc.U: Call ./whoa
-	explicitely instead of relying on PATH.
+	explicitly instead of relying on PATH.
 
 	* mcon/U/Loc.U: Added ?F: line for metalint file checking.  Be
 	careful and guard against wildcard searching (ADO).
@@ -1323,10 +1323,10 @@ Sat Oct 29 18:16:03 MET 1994   Raphael Manfredi <ram@acri.fr>
 	* mcon/U/Typedef.U: Don't clobber visible 'val' variable, use
 	'varval' instead.
 
-	* mcon/U/Warn_v7EXT.U, mcon/U/Warn_v7ND.U: Call ./v7 explicitely
+	* mcon/U/Warn_v7EXT.U, mcon/U/Warn_v7ND.U: Call ./v7 explicitly
 	instead of relying on PATH.
 
-	* mcon/U/abortsig.U, mcon/U/d_bsdjmp.U: Call ./usg explicitely
+	* mcon/U/abortsig.U, mcon/U/d_bsdjmp.U: Call ./usg explicitly
 	instead of relying on PATH.
 
 	* mcon/U/d_attribut.U, mcon/U/d_gconvert.U, mcon/U/d_drem.U,
@@ -1358,19 +1358,19 @@ Sat Oct 29 18:16:03 MET 1994   Raphael Manfredi <ram@acri.fr>
 	McQueen).
 
 	* mcon/U/d_keepsig.U, mcon/U/d_eofpipe.U, mcon/U/d_normsig.U,
-	mcon/U/d_tzmin.U: Call ./bsd explicitely instead of relying on PATH.
+	mcon/U/d_tzmin.U: Call ./bsd explicitly instead of relying on PATH.
 
-	* mcon/U/d_gethname.U: Call ./xenix explicitely instead of relying on
+	* mcon/U/d_gethname.U: Call ./xenix explicitly instead of relying on
 	PATH.
 
-	* mcon/U/d_newsadm.U: Call ./eunice explicitely instead of relying on
+	* mcon/U/d_newsadm.U: Call ./eunice explicitly instead of relying on
 	PATH.
 
 	* mcon/U/d_setpgrp.U: Added 'ldflags' to the test compile line (ADO).
-	Call ./usg explicitely instead of relying on PATH.
+	Call ./usg explicitly instead of relying on PATH.
 
 	* mcon/U/registers.U, mcon/U/shm_for.U, mcon/U/d_speedopt.U: Call
-	./Cppsym explicitely instead of relying on PATH.
+	./Cppsym explicitly instead of relying on PATH.
 
 	* mcon/U/d_time.U: Now uses new Typedef unit to compute type
 	information (ADO).
@@ -1384,7 +1384,7 @@ Sat Oct 29 18:16:03 MET 1994   Raphael Manfredi <ram@acri.fr>
 	* mcon/U/i_neterrno.U: Forgot to initialize 'val' to an empty value
 	(ADO).
 
-	* mcon/U/i_termio.U: Call ./usg and ./Cppsym explicitely instead of
+	* mcon/U/i_termio.U: Call ./usg and ./Cppsym explicitly instead of
 	relying on PATH.
 
 	* mcon/U/lex.U: Spurious single quote could cause Configure to crash.
@@ -1398,10 +1398,10 @@ Sat Oct 29 18:16:03 MET 1994   Raphael Manfredi <ram@acri.fr>
 	* mcon/U/mailfile.U: The Loc unit was missing from the dependency
 	line.
 
-	* mcon/U/myhostname.U: Call ./xenix explicitely instead of relying on
+	* mcon/U/myhostname.U: Call ./xenix explicitly instead of relying on
 	PATH.  Now uses new Tr unit to convert to/from lowercase.
 
-	* mcon/U/nametype.U: Call ./usg and ./bsd explicitely instead of
+	* mcon/U/nametype.U: Call ./usg and ./bsd explicitly instead of
 	relying on PATH.
 
 	* mcon/U/orderlib.U: Now performs a real small compile for accurate
diff --git a/MANIFEST b/MANIFEST
index ffd22b5..d3d2050 100644
--- a/MANIFEST
+++ b/MANIFEST
@@ -150,7 +150,7 @@ mcon/U/cc.U                Which C compiler should be used?
 mcon/U/ccflags.U           Any additional C flags?
 mcon/U/cf_email.U          Computes user's e-mail address
 mcon/U/cf_name.U           Computes user's full name
-mcon/U/cf_who.U            Who configured the sytem, where and when
+mcon/U/cf_who.U            Who configured the system, where and when
 mcon/U/charorder.U         Character order within a byte
 mcon/U/charsize.U          What is the size of "char"?
 mcon/U/contains.U          Makes status-returning grep
diff --git a/bin/manicheck.man b/bin/manicheck.man
index 3419a03..96c71ab 100644
--- a/bin/manicheck.man
+++ b/bin/manicheck.man
@@ -1,18 +1,18 @@
-''' $Id: manicheck.man 3 2006-08-25 21:39:07Z rmanfredi $
-'''
-'''  Copyright (c) 1991-1997, 2004-2006, Raphael Manfredi
-'''  
-'''  You may redistribute only under the terms of the Artistic Licence,
-'''  as specified in the README file that comes with the distribution.
-'''  You may reuse parts of this distribution only within the terms of
-'''  that same Artistic Licence; a copy of which may be found at the root
-'''  of the source tree for dist 4.0.
-'''
-''' $Log: manicheck.man,v $
-''' Revision 3.0  1993/08/18  12:04:02  ram
-''' Baseline for dist 3.0 netwide release.
-'''
-''' 
+.\" $Id: manicheck.man 3 2006-08-25 21:39:07Z rmanfredi $
+.\"
+.\"  Copyright (c) 1991-1997, 2004-2006, Raphael Manfredi
+.\"  
+.\"  You may redistribute only under the terms of the Artistic Licence,
+.\"  as specified in the README file that comes with the distribution.
+.\"  You may reuse parts of this distribution only within the terms of
+.\"  that same Artistic Licence; a copy of which may be found at the root
+.\"  of the source tree for dist 4.0.
+.\"
+.\" $Log: manicheck.man,v $
+.\" Revision 3.0  1993/08/18  12:04:02  ram
+.\" Baseline for dist 3.0 netwide release.
+.\"
+.\" 
 .TH MANICHECK 1 ram
 .SH NAME
 manicheck \- check manifest accuracy
diff --git a/bin/manilist.man b/bin/manilist.man
index a2efb86..5d51c53 100644
--- a/bin/manilist.man
+++ b/bin/manilist.man
@@ -1,27 +1,27 @@
-''' $Id: manilist.man 3 2006-08-25 21:39:07Z rmanfredi $
-'''
-'''  Copyright (c) 1991-1997, 2004-2006, Raphael Manfredi
-'''  
-'''  You may redistribute only under the terms of the Artistic Licence,
-'''  as specified in the README file that comes with the distribution.
-'''  You may reuse parts of this distribution only within the terms of
-'''  that same Artistic Licence; a copy of which may be found at the root
-'''  of the source tree for dist 4.0.
-'''
-''' $Log: manilist.man,v $
-''' Revision 3.0.1.3  1995/05/12  11:57:31  ram
-''' patch54: updated my e-mail address
-'''
-''' Revision 3.0.1.2  1994/01/24  13:52:55  ram
-''' patch16: typo fix
-'''
-''' Revision 3.0.1.1  1993/08/24  12:11:02  ram
-''' patch3: typo fix
-'''
-''' Revision 3.0  1993/08/18  12:04:04  ram
-''' Baseline for dist 3.0 netwide release.
-'''
-''' 
+.\" $Id: manilist.man 3 2006-08-25 21:39:07Z rmanfredi $
+.\"
+.\"  Copyright (c) 1991-1997, 2004-2006, Raphael Manfredi
+.\"  
+.\"  You may redistribute only under the terms of the Artistic Licence,
+.\"  as specified in the README file that comes with the distribution.
+.\"  You may reuse parts of this distribution only within the terms of
+.\"  that same Artistic Licence; a copy of which may be found at the root
+.\"  of the source tree for dist 4.0.
+.\"
+.\" $Log: manilist.man,v $
+.\" Revision 3.0.1.3  1995/05/12  11:57:31  ram
+.\" patch54: updated my e-mail address
+.\"
+.\" Revision 3.0.1.2  1994/01/24  13:52:55  ram
+.\" patch16: typo fix
+.\"
+.\" Revision 3.0.1.1  1993/08/24  12:11:02  ram
+.\" patch3: typo fix
+.\"
+.\" Revision 3.0  1993/08/18  12:04:04  ram
+.\" Baseline for dist 3.0 netwide release.
+.\"
+.\" 
 .TH MANILIST 1 ram
 .SH NAME
 manilist \- report status of files in a source directory
diff --git a/bin/packinit.SH b/bin/packinit.SH
index d0e6d9f..aed01b3 100755
--- a/bin/packinit.SH
+++ b/bin/packinit.SH
@@ -16,6 +16,7 @@ case "$0" in
 esac
 echo "Extracting bin/packinit (with variable substitutions)"
 cat >packinit <<!GROK!THIS!
+#! /usr/bin/perl
 # feed this into perl
 	eval 'exec perl -S \$0 \${1+"\$@"}'
 		if \$running_under_some_shell;
diff --git a/bin/packinit.man b/bin/packinit.man
index 97fe728..bfadfe4 100644
--- a/bin/packinit.man
+++ b/bin/packinit.man
@@ -1,30 +1,30 @@
-''' $Id: packinit.man 3 2006-08-25 21:39:07Z rmanfredi $
-'''
-'''  Copyright (c) 1991-1997, 2004-2006, Raphael Manfredi
-'''  
-'''  You may redistribute only under the terms of the Artistic Licence,
-'''  as specified in the README file that comes with the distribution.
-'''  You may reuse parts of this distribution only within the terms of
-'''  that same Artistic Licence; a copy of which may be found at the root
-'''  of the source tree for dist 4.0.
-'''
-''' $Log: packinit.man,v $
-''' Revision 3.0.1.4  1995/07/25  13:31:38  ram
-''' patch56: fixed a typo
-'''
-''' Revision 3.0.1.3  1995/05/12  11:57:38  ram
-''' patch54: updated my e-mail address
-'''
-''' Revision 3.0.1.2  1994/10/29  15:45:17  ram
-''' patch36: added new variables cext, shext, changelog and changercs
-'''
-''' Revision 3.0.1.1  1994/01/24  13:54:31  ram
-''' patch16: now documents variables set in .package by packinit
-'''
-''' Revision 3.0  1993/08/18  12:04:06  ram
-''' Baseline for dist 3.0 netwide release.
-'''
-''' 
+.\" $Id: packinit.man 3 2006-08-25 21:39:07Z rmanfredi $
+.\"
+.\"  Copyright (c) 1991-1997, 2004-2006, Raphael Manfredi
+.\"  
+.\"  You may redistribute only under the terms of the Artistic Licence,
+.\"  as specified in the README file that comes with the distribution.
+.\"  You may reuse parts of this distribution only within the terms of
+.\"  that same Artistic Licence; a copy of which may be found at the root
+.\"  of the source tree for dist 4.0.
+.\"
+.\" $Log: packinit.man,v $
+.\" Revision 3.0.1.4  1995/07/25  13:31:38  ram
+.\" patch56: fixed a typo
+.\"
+.\" Revision 3.0.1.3  1995/05/12  11:57:38  ram
+.\" patch54: updated my e-mail address
+.\"
+.\" Revision 3.0.1.2  1994/10/29  15:45:17  ram
+.\" patch36: added new variables cext, shext, changelog and changercs
+.\"
+.\" Revision 3.0.1.1  1994/01/24  13:54:31  ram
+.\" patch16: now documents variables set in .package by packinit
+.\"
+.\" Revision 3.0  1993/08/18  12:04:06  ram
+.\" Baseline for dist 3.0 netwide release.
+.\"
+.\" 
 .TH PACKINIT 1 ram
 .SH NAME
 packinit \- initialize or update your .package file
@@ -145,4 +145,4 @@ Larry Wall <lwall@netlabs.com> (version 2.0)
 .br
 Raphael Manfredi <ram@hptnos02.grenoble.hp.com>
 .SH "SEE ALSO"
-dist(1).
+dist(7).
diff --git a/bin/perload b/bin/perload
index 4dc77c2..e9e5544 100755
--- a/bin/perload
+++ b/bin/perload
@@ -1,4 +1,4 @@
-: # feed this into perl
+#! /usr/bin/perl
 '/bin/true' && eval 'exec perl -S $0 "$@"'
 	if $running_under_some_shell;
 'di';
@@ -478,9 +478,9 @@ sub q {
 .nr % 0		\" start at page 1
 '; __END__	\" the perl compiler stops here
 
-'''
-''' From here on it's a standard manual page.
-'''
+.\"
+.\" From here on it's a standard manual page.
+.\"
 
 .TH PERLOAD 1 "June 20, 1992"
 .AT 3
diff --git a/dist.man b/dist.man
index ebb8ccb..da0e60e 100644
--- a/dist.man
+++ b/dist.man
@@ -1,37 +1,37 @@
-''' $Id: dist.man 3 2006-08-25 21:39:07Z rmanfredi $
-'''
-'''  Copyright (c) 1991-1997, 2004-2006, Raphael Manfredi
-'''  
-'''  You may redistribute only under the terms of the Artistic Licence,
-'''  as specified in the README file that comes with the distribution.
-'''  You may reuse parts of this distribution only within the terms of
-'''  that same Artistic Licence; a copy of which may be found at the root
-'''  of the source tree for dist 4.0.
-'''
-''' $Log: dist.man,v $
-''' Revision 3.0.1.6  1995/05/12  11:57:53  ram
-''' patch54: updated my e-mail address
-'''
-''' Revision 3.0.1.5  1994/10/29  15:46:03  ram
-''' patch36: mentions new patlog script and ChangeLog file
-'''
-''' Revision 3.0.1.4  1994/05/06  13:54:17  ram
-''' patch23: extended copyright notice to 1994
-''' patch23: new script kitpost
-'''
-''' Revision 3.0.1.3  1994/01/24  13:55:41  ram
-''' patch16: documents profile and its components
-'''
-''' Revision 3.0.1.2  1993/11/10  17:31:03  ram
-''' patch14: added mention for new confmagic.h file
-'''
-''' Revision 3.0.1.1  1993/08/24  12:12:00  ram
-''' patch3: added entries for patnotify and patsnap
-'''
-''' Revision 3.0  1993/08/18  12:04:07  ram
-''' Baseline for dist 3.0 netwide release.
-'''
-''' 
+.\" $Id: dist.man 3 2006-08-25 21:39:07Z rmanfredi $
+.\"
+.\"  Copyright (c) 1991-1997, 2004-2006, Raphael Manfredi
+.\"  
+.\"  You may redistribute only under the terms of the Artistic Licence,
+.\"  as specified in the README file that comes with the distribution.
+.\"  You may reuse parts of this distribution only within the terms of
+.\"  that same Artistic Licence; a copy of which may be found at the root
+.\"  of the source tree for dist 4.0.
+.\"
+.\" $Log: dist.man,v $
+.\" Revision 3.0.1.6  1995/05/12  11:57:53  ram
+.\" patch54: updated my e-mail address
+.\"
+.\" Revision 3.0.1.5  1994/10/29  15:46:03  ram
+.\" patch36: mentions new patlog script and ChangeLog file
+.\"
+.\" Revision 3.0.1.4  1994/05/06  13:54:17  ram
+.\" patch23: extended copyright notice to 1994
+.\" patch23: new script kitpost
+.\"
+.\" Revision 3.0.1.3  1994/01/24  13:55:41  ram
+.\" patch16: documents profile and its components
+.\"
+.\" Revision 3.0.1.2  1993/11/10  17:31:03  ram
+.\" patch14: added mention for new confmagic.h file
+.\"
+.\" Revision 3.0.1.1  1993/08/24  12:12:00  ram
+.\" patch3: added entries for patnotify and patsnap
+.\"
+.\" Revision 3.0  1993/08/18  12:04:07  ram
+.\" Baseline for dist 3.0 netwide release.
+.\"
+.\" 
 .de Ex		\" Start of Example
 .sp
 .in +5
@@ -42,7 +42,7 @@
 .in -5
 .fi
 ..
-.TH DIST 1 ram
+.TH DIST 7 ram
 .SH NAME
 dist \- introduction to dist
 .SH DESCRIPTION
diff --git a/jmake/bindex.SH b/jmake/bindex.SH
index ca73f5e..ee45b2e 100755
--- a/jmake/bindex.SH
+++ b/jmake/bindex.SH
@@ -20,7 +20,7 @@ $startperl
 !GROK!THIS!
 $spitshell >>bindex <<'!NO!SUBS!'
 	eval 'exec perl -S $0 ${1+"$@"}'
-		if $runnning_under_some_shell;
+		if $running_under_some_shell;
 
 # $Id: bindex.SH 31 2010-03-06 20:15:24Z rmanfredi $
 #
diff --git a/jmake/files/Jmake.rules b/jmake/files/Jmake.rules
index 8877f17..0b0c02d 100644
--- a/jmake/files/Jmake.rules
+++ b/jmake/files/Jmake.rules
@@ -512,7 +512,7 @@ local_deinstall.man::		@@\
  */
 #define LinkFileList(step,list,dir,sub)	@!\
 step:: list							@@\
-	@case '${MFLAGS}' in *[i]*) set +e;; esac; \	@@\
+	@case '${MFLAGS}' in *[i]*) set +e;; *) set -e;; esac; \	@@\
 	echo "	cd" dir; cd dir; for i in list; do (set -x; $(LN) sub/$$i .); done
 
 
@@ -1191,7 +1191,7 @@ local_realclean:: 								@@\
  */
 #define CommonSubdirsRule(dirs) @!\
 subdirs: 						@@\
-	@case '${MFLAGS}' in *[ik]*) set +e;; esac; \			@@\
+	@case '${MFLAGS}' in *[ik]*) set +e;; *) set -e;; esac; \		@@\
 	for i in dirs ;\					@@\
 	do \								@@\
 		(cd $$i ; echo $(VERB) "in $(DIR)$$i..."; \		@@\
@@ -1205,7 +1205,7 @@ subdirs: 						@@\
  */
 #define NamedTargetSubdirsRule(dirs,name,verb,flags)	@!\
 name::									@@\
-	@case '${MFLAGS}' in *[ik]*) set +e;; esac; \		@@\
+	@case '${MFLAGS}' in *[ik]*) set +e;; *) set -e;; esac; \		@@\
 	for i in dirs ;\					@@\
 	do \								@@\
 		(cd $$i ; echo verb "in $(DIR)$$i..."; \	@@\
@@ -1449,7 +1449,7 @@ MakeLintSubdirs(dirs,lintlib)
  */
 #define MakeMakeSubdirs(target)	@!\
 target::								@@\
-	@case '${MFLAGS}' in *[ik]*) set +e;; esac; \	@@\
+	@case '${MFLAGS}' in *[ik]*) set +e;; *) set -e;; esac; \	@@\
 	for i in $(SUBDIRS);\				@@\
 	do \								@@\
 		echo "Making "target" in $(DIR)$$i..."; \	@@\
@@ -1468,7 +1468,7 @@ target::								@@\
  */
 #define MakeMakefilesSH()		@!\
 Makefiles.SH:: Makefile.SH				@@\
-	@case '${MFLAGS}' in *[ik]*) set +e;; esac; \	@@\
+	@case '${MFLAGS}' in *[ik]*) set +e;; *) set -e;; esac; \	@@\
 	for i in $(SUBDIRS);\					@@\
 	do \								@@\
 		case "$(DIR)$$i/" in \								@@\
diff --git a/jmake/fixcpp.SH b/jmake/fixcpp.SH
index 092a4ca..fa816f4 100755
--- a/jmake/fixcpp.SH
+++ b/jmake/fixcpp.SH
@@ -19,7 +19,7 @@ echo "Extracting jmake/fixcpp (with variable substitutions)"
 $spitshell >fixcpp <<!GROK!THIS!
 $startperl
 	eval 'exec perl -S \$0 \${1+"\$@"}'
-		if \$runnning_under_some_shell;
+		if \$running_under_some_shell;
 
 # $Id: fixcpp.SH 42 2010-11-28 18:15:13Z rmanfredi $
 #
diff --git a/jmake/jmake.SH b/jmake/jmake.SH
index 392952f..6f97180 100755
--- a/jmake/jmake.SH
+++ b/jmake/jmake.SH
@@ -19,7 +19,7 @@ echo "Extracting jmake/jmake (with variable substitutions)"
 $spitshell >jmake <<!GROK!THIS!
 $startperl
 	eval 'exec perl -S \$0 \${1+"\$@"}'
-		if \$runnning_under_some_shell;
+		if \$running_under_some_shell;
 
 # $Id: jmake.SH 47 2010-11-28 22:23:13Z rmanfredi $
 #
diff --git a/jmake/jmake.man b/jmake/jmake.man
index afd5e65..5f577e6 100644
--- a/jmake/jmake.man
+++ b/jmake/jmake.man
@@ -1,28 +1,28 @@
-''' $Id: jmake.man 47 2010-11-28 22:23:13Z rmanfredi $
-'''
-'''  Copyright (c) 1991-1997, 2004-2006, Raphael Manfredi
-'''  
-'''  You may redistribute only under the terms of the Artistic Licence,
-'''  as specified in the README file that comes with the distribution.
-'''  You may reuse parts of this distribution only within the terms of
-'''  that same Artistic Licence; a copy of which may be found at the root
-'''  of the source tree for dist 4.0.
-'''
-''' $Log: jmake.man,v $
-''' Revision 3.0.1.3  2004/08/22 09:01:55  ram
-''' patch71: renamed |test as |case as the construct has its syntax
-''' patch71: added |subst section to allow variable substitutions
-'''
-''' Revision 3.0.1.2  2004/08/21 23:19:52  ram
-''' patch71: added '|shell' section to emit verbatim code in Makefile.SH
-''' patch71: new '|test' to conditionally generate Makefile sections
-'''
-''' Revision 3.0.1.1  1995/05/12  11:57:58  ram
-''' patch54: updated my e-mail address
-'''
-''' Revision 3.0  1993/08/18  12:04:18  ram
-''' Baseline for dist 3.0 netwide release.
-'''
+.\" $Id: jmake.man 47 2010-11-28 22:23:13Z rmanfredi $
+.\"
+.\"  Copyright (c) 1991-1997, 2004-2006, Raphael Manfredi
+.\"
+.\"  You may redistribute only under the terms of the Artistic Licence,
+.\"  as specified in the README file that comes with the distribution.
+.\"  You may reuse parts of this distribution only within the terms of
+.\"  that same Artistic Licence; a copy of which may be found at the root
+.\"  of the source tree for dist 4.0.
+.\"
+.\" $Log: jmake.man,v $
+.\" Revision 3.0.1.3  2004/08/22 09:01:55  ram
+.\" patch71: renamed |test as |case as the construct has its syntax
+.\" patch71: added |subst section to allow variable substitutions
+.\"
+.\" Revision 3.0.1.2  2004/08/21 23:19:52  ram
+.\" patch71: added '|shell' section to emit verbatim code in Makefile.SH
+.\" patch71: new '|test' to conditionally generate Makefile sections
+.\"
+.\" Revision 3.0.1.1  1995/05/12  11:57:58  ram
+.\" patch54: updated my e-mail address
+.\"
+.\" Revision 3.0  1993/08/18  12:04:18  ram
+.\" Baseline for dist 3.0 netwide release.
+.\"
 .TH JMAKE 1 ram
 .SH NAME
 jmake \- a generic makefile builder
@@ -195,16 +195,16 @@ Available commands are:
 |rule: <text>: same as before, with a leading tab.
 .sp
 .IP \(bu
-|skip: skips text until a line starting with '-skip' is found.
+|skip: skips text until a line starting with '\-skip' is found.
 .sp
 .IP \(bu
 |subst: begins section where lines will be subject to variable substitution,
-until '-subst' is found.  This means that when the Makefile.SH is run, all
+until '\-subst' is found.  This means that when the Makefile.SH is run, all
 instances of $var within the \fIsubst\fR section will be substituted by the
 shell.
 .sp
 .IP \(bu
-|shell: emits section until matching '-shell' as-is in the generated
+|shell: emits section until matching '\-shell' as-is in the generated
 Makefile.SH.  This can be useful to prepare \fI|case\fR sections.  It is
 not allowed to nest \fIshell\fR sections.
 .sp
@@ -213,15 +213,15 @@ not allowed to nest \fIshell\fR sections.
 its leading '$' sign) and a case-style pattern, for instance the string
 "var in f*".  It will generate the corresponding "case" test in the
 Makefile.SH on the "$var" value and only if this test is true will the
-section until the matching '-case' be generated in the Makefile when
+section until the matching '\-case' be generated in the Makefile when
 Makefile.SH is run.  It is possible to nest \fIcase\fR sections freely.
 .sp
 .IP \(bu
-|expand <pattern>: expand lines until '-expand' with <pattern>. A
+|expand <pattern>: expand lines until '\-expand' with <pattern>. A
 complete example is shown below.
 .sp
 .IP \(bu
-|once <symbol>: text up to '-once' appears only the first time.
+|once <symbol>: text up to '\-once' appears only the first time.
 .sp
 .PP
 The '|' commands cannot be nested, unless otherwise noted.
@@ -239,7 +239,7 @@ Here is a way to implement a logical OR:
 ?SYMBOL:text		/* Keeps text if SYMBOL */
 %SYMBOL:|skip
 	%TOKEN:text		/* Keeps text if not TOKEN */
--skip
+\-skip
 .fi
 .in -5
 .sp
@@ -256,7 +256,7 @@ not (p or q) <=> not p and not q
 /* Implements SYMBOL or not TOKEN (attempt #2) */
 %SYMBOL:?TOKEN:|skip
 text					/* If SYMBOL or not TOKEN */
--skip
+\-skip
 .sp
 .in -5
 .fi
@@ -273,7 +273,7 @@ an example first:
 |expand a!foo bar! b!yes no!
 !a::
 	echo !a, !b
--expand
+\-expand
 .fi
 .in -5
 .sp
@@ -302,7 +302,7 @@ It is defined in \fIJmake.rules\fR as:
 #define Expand(rule, pattern) @!\\
 |expand pattern @!\\
 rule @!\\
--expand
+\-expand
 .sp
 .in -5
 .fi
@@ -316,7 +316,7 @@ A = foo bar
 
 #define Rule @!\\
 $(DIR)/!a^^.o: !a^^.o @@\\
-	$(CC) -c !a^^.c @@\\
+	$(CC) \-c !a^^.c @@\\
 	$(MV) !a^^.o $(DIR)
 
 Expand(Rule, a!$(A)!)
@@ -328,11 +328,11 @@ which will generate in \fIMakefile.SH\fR:
 .in +5
 .nf
 $(DIR)/foo.o: foo.o
-	$(CC) -c foo.c
+	$(CC) \-c foo.c
 	$(MV) foo.o $(DIR)
 
 $(DIR)/bar.o: bar.o
-	$(CC) -c bar.c
+	$(CC) \-c bar.c
 	$(MV) bar.o $$(DIR)
 .sp
 .in -5
@@ -349,7 +349,7 @@ values for the \fBfirst\fR parameter. If other parameters have less
 substitution values, they will get void ones.
 .sp
 .PP
-It is possible to add a regular expression at the end of '-expand'. This
+It is possible to add a regular expression at the end of '\-expand'. This
 regular expression will be removed from the final set of expansion at the
 end of each line. It is also possible to do substitutions in the expanded
 item, by using the syntax (if 'f' is the expanded variable)
@@ -371,11 +371,11 @@ SRC = foo.c bar.c
 OBJ = \\
 |expand f!$(SRC)!
 	!f:\\.c=\\.o \\
--expand \\\\
+\-expand \\\\
 INC = \\
 |expand f!$(OBJ)!
 	!f:\\.o=\\.h \\
--expand \\\\
+\-expand \\\\
 .fi
 .in -5
 .sp
@@ -398,7 +398,7 @@ as backslash, point, etc...
 .sp
 .PP
 The \fIonce\fR command is tagged with a name. The first time the name
-appears, the once construct is ignored and the text up to '-once' will
+appears, the once construct is ignored and the text up to '\-once' will
 be copied in the generated Makefile.SH.  However, future occurences of
 the same name will be ignored (\fIonce\fR will behave like \fIskip\fR).
 .sp
@@ -408,7 +408,7 @@ Example:
 .nf
 |once this_is_a_name
 <text>
--once
+\-once
 .sp
 .in -5
 .fi
@@ -423,7 +423,7 @@ in the Makefile.SH.  For instance, the following section in the Jmakefile:
 case "$d_usegtk1" in
 define) glib=1; gtk=1;;
 esac
--shell
+\-shell
 .sp
 .in -5
 .fi
@@ -440,7 +440,7 @@ parts of the Makefile only when building for GTK1 using glib-1.x:
 |case glib in 1
 display:
 	echo "Building for glib-1.x"
--case
+\-case
 .sp
 .in -5
 .fi
@@ -482,7 +482,7 @@ User-defined variables:
 .sp
 The user may define CFLAGS, LDFLAGS or DPFLAGS as additional flags to be used
 in C compilation, linking phase or depend target. It is thus possible to add
-some extra flags such as -I or libraries for Makefiles in specific
+some extra flags such as \-I or libraries for Makefiles in specific
 sub-directories.
 .sp
 .PD
diff --git a/jmake/jmkmf.man b/jmake/jmkmf.man
index 60595e3..4411f2f 100644
--- a/jmake/jmkmf.man
+++ b/jmake/jmkmf.man
@@ -1,20 +1,20 @@
-''' $Id: jmkmf.man 3 2006-08-25 21:39:07Z rmanfredi $
-'''
-'''  Copyright (c) 1991-1997, 2004-2006, Raphael Manfredi
-'''  
-'''  You may redistribute only under the terms of the Artistic Licence,
-'''  as specified in the README file that comes with the distribution.
-'''  You may reuse parts of this distribution only within the terms of
-'''  that same Artistic Licence; a copy of which may be found at the root
-'''  of the source tree for dist 4.0.
-'''
-''' $Log: jmkmf.man,v $
-''' Revision 3.0.1.1  1995/05/12  11:58:03  ram
-''' patch54: updated my e-mail address
-'''
-''' Revision 3.0  1993/08/18  12:04:20  ram
-''' Baseline for dist 3.0 netwide release.
-'''
+.\" $Id: jmkmf.man 3 2006-08-25 21:39:07Z rmanfredi $
+.\"
+.\"  Copyright (c) 1991-1997, 2004-2006, Raphael Manfredi
+.\"  
+.\"  You may redistribute only under the terms of the Artistic Licence,
+.\"  as specified in the README file that comes with the distribution.
+.\"  You may reuse parts of this distribution only within the terms of
+.\"  that same Artistic Licence; a copy of which may be found at the root
+.\"  of the source tree for dist 4.0.
+.\"
+.\" $Log: jmkmf.man,v $
+.\" Revision 3.0.1.1  1995/05/12  11:58:03  ram
+.\" patch54: updated my e-mail address
+.\"
+.\" Revision 3.0  1993/08/18  12:04:20  ram
+.\" Baseline for dist 3.0 netwide release.
+.\"
 .TH JMKMF 1 ram
 .SH NAME
 jmkmf \- runs jmake with the correct options
diff --git a/kit/kitpost.man b/kit/kitpost.man
index e840525..6777e5e 100644
--- a/kit/kitpost.man
+++ b/kit/kitpost.man
@@ -1,20 +1,20 @@
-''' $Id: kitpost.man 3 2006-08-25 21:39:07Z rmanfredi $
-'''
-'''  Copyright (c) 1991-1997, 2004-2006, Raphael Manfredi
-'''  
-'''  You may redistribute only under the terms of the Artistic Licence,
-'''  as specified in the README file that comes with the distribution.
-'''  You may reuse parts of this distribution only within the terms of
-'''  that same Artistic Licence; a copy of which may be found at the root
-'''  of the source tree for dist 4.0.
-'''
-''' $Log: kitpost.man,v $
-''' Revision 3.0.1.2  1995/05/12  11:58:09  ram
-''' patch54: updated my e-mail address
-'''
-''' Revision 3.0.1.1  1994/05/06  13:55:01  ram
-''' patch23: created
-'''
+.\" $Id: kitpost.man 3 2006-08-25 21:39:07Z rmanfredi $
+.\"
+.\"  Copyright (c) 1991-1997, 2004-2006, Raphael Manfredi
+.\"  
+.\"  You may redistribute only under the terms of the Artistic Licence,
+.\"  as specified in the README file that comes with the distribution.
+.\"  You may reuse parts of this distribution only within the terms of
+.\"  that same Artistic Licence; a copy of which may be found at the root
+.\"  of the source tree for dist 4.0.
+.\"
+.\" $Log: kitpost.man,v $
+.\" Revision 3.0.1.2  1995/05/12  11:58:09  ram
+.\" patch54: updated my e-mail address
+.\"
+.\" Revision 3.0.1.1  1994/05/06  13:55:01  ram
+.\" patch23: created
+.\"
 .TH KITSEND 1 ram
 .SH NAME
 kitpost \- posts distribution kits
diff --git a/kit/kitsend.man b/kit/kitsend.man
index 0bcfb0f..3915dd7 100644
--- a/kit/kitsend.man
+++ b/kit/kitsend.man
@@ -1,20 +1,20 @@
-''' $Id: kitsend.man 3 2006-08-25 21:39:07Z rmanfredi $
-'''
-'''  Copyright (c) 1991-1997, 2004-2006, Raphael Manfredi
-'''  
-'''  You may redistribute only under the terms of the Artistic Licence,
-'''  as specified in the README file that comes with the distribution.
-'''  You may reuse parts of this distribution only within the terms of
-'''  that same Artistic Licence; a copy of which may be found at the root
-'''  of the source tree for dist 4.0.
-'''
-''' $Log: kitsend.man,v $
-''' Revision 3.0.1.1  1994/05/06  14:00:11  ram
-''' patch23: documented new -V and -h options
-'''
-''' Revision 3.0  1993/08/18  12:04:26  ram
-''' Baseline for dist 3.0 netwide release.
-'''
+.\" $Id: kitsend.man 3 2006-08-25 21:39:07Z rmanfredi $
+.\"
+.\"  Copyright (c) 1991-1997, 2004-2006, Raphael Manfredi
+.\"  
+.\"  You may redistribute only under the terms of the Artistic Licence,
+.\"  as specified in the README file that comes with the distribution.
+.\"  You may reuse parts of this distribution only within the terms of
+.\"  that same Artistic Licence; a copy of which may be found at the root
+.\"  of the source tree for dist 4.0.
+.\"
+.\" $Log: kitsend.man,v $
+.\" Revision 3.0.1.1  1994/05/06  14:00:11  ram
+.\" patch23: documented new -V and -h options
+.\"
+.\" Revision 3.0  1993/08/18  12:04:26  ram
+.\" Baseline for dist 3.0 netwide release.
+.\"
 .TH KITSEND 1 ram
 .SH NAME
 kitsend \- sends distribution kits
diff --git a/kit/makeSH.man b/kit/makeSH.man
index 5995cfe..ef8ed89 100644
--- a/kit/makeSH.man
+++ b/kit/makeSH.man
@@ -1,18 +1,18 @@
-''' $Id: makeSH.man 3 2006-08-25 21:39:07Z rmanfredi $
-'''
-'''  Copyright (c) 1991-1997, 2004-2006, Raphael Manfredi
-'''  
-'''  You may redistribute only under the terms of the Artistic Licence,
-'''  as specified in the README file that comes with the distribution.
-'''  You may reuse parts of this distribution only within the terms of
-'''  that same Artistic Licence; a copy of which may be found at the root
-'''  of the source tree for dist 4.0.
-'''
-''' $Log: makeSH.man,v $
-''' Revision 3.0  1993/08/18  12:04:27  ram
-''' Baseline for dist 3.0 netwide release.
-'''
-''' 
+.\" $Id: makeSH.man 3 2006-08-25 21:39:07Z rmanfredi $
+.\"
+.\"  Copyright (c) 1991-1997, 2004-2006, Raphael Manfredi
+.\"  
+.\"  You may redistribute only under the terms of the Artistic Licence,
+.\"  as specified in the README file that comes with the distribution.
+.\"  You may reuse parts of this distribution only within the terms of
+.\"  that same Artistic Licence; a copy of which may be found at the root
+.\"  of the source tree for dist 4.0.
+.\"
+.\" $Log: makeSH.man,v $
+.\" Revision 3.0  1993/08/18  12:04:27  ram
+.\" Baseline for dist 3.0 netwide release.
+.\"
+.\" 
 .TH MAKESH 1 LOCAL
 .SH NAME
 makeSH \- a .SH script maker
diff --git a/kit/makedist.man b/kit/makedist.man
index 8312a92..52ffa3d 100644
--- a/kit/makedist.man
+++ b/kit/makedist.man
@@ -1,25 +1,25 @@
 .rn '' }`
-''' $Id: makedist.man 3 2006-08-25 21:39:07Z rmanfredi $
-'''
-'''  Copyright (c) 1991-1997, 2004-2006, Raphael Manfredi
-'''  
-'''  You may redistribute only under the terms of the Artistic Licence,
-'''  as specified in the README file that comes with the distribution.
-'''  You may reuse parts of this distribution only within the terms of
-'''  that same Artistic Licence; a copy of which may be found at the root
-'''  of the source tree for dist 4.0.
-'''
-''' $Log: makedist.man,v $
-''' Revision 3.0.1.2  1995/05/12  11:58:16  ram
-''' patch54: updated my e-mail address
-'''
-''' Revision 3.0.1.1  1994/05/06  14:00:50  ram
-''' patch23: now mentions kitpost and kitsend
-'''
-''' Revision 3.0  1993/08/18  12:04:31  ram
-''' Baseline for dist 3.0 netwide release.
-'''
-''' 
+.\" $Id: makedist.man 3 2006-08-25 21:39:07Z rmanfredi $
+.\"
+.\"  Copyright (c) 1991-1997, 2004-2006, Raphael Manfredi
+.\"  
+.\"  You may redistribute only under the terms of the Artistic Licence,
+.\"  as specified in the README file that comes with the distribution.
+.\"  You may reuse parts of this distribution only within the terms of
+.\"  that same Artistic Licence; a copy of which may be found at the root
+.\"  of the source tree for dist 4.0.
+.\"
+.\" $Log: makedist.man,v $
+.\" Revision 3.0.1.2  1995/05/12  11:58:16  ram
+.\" patch54: updated my e-mail address
+.\"
+.\" Revision 3.0.1.1  1994/05/06  14:00:50  ram
+.\" patch23: now mentions kitpost and kitsend
+.\"
+.\" Revision 3.0  1993/08/18  12:04:31  ram
+.\" Baseline for dist 3.0 netwide release.
+.\"
+.\" 
 .de Sh
 .br
 .ne 5
@@ -31,11 +31,11 @@
 .if t .sp .5v
 .if n .sp
 ..
-'''
-'''     Set up \*(-- to give an unbreakable dash;
-'''     string Tr holds user defined translation string.
-'''     Bell System Logo is used as a dummy character.
-'''
+.\"
+.\"     Set up \*(-- to give an unbreakable dash;
+.\"     string Tr holds user defined translation string.
+.\"     Bell System Logo is used as a dummy character.
+.\"
 .ie n \{\
 .tr \(*W-\*(Tr
 .ds -- \(*W-
@@ -100,7 +100,7 @@ automatically split in smaller parts. Only the first 11 characters of the file
 will be kept though, and \fImakedist\fR will abort if two distinct files are
 to be split and have the same 11 first characters in their names. The split
 files will automatically be reconstructed at the end of the archive extraction
-by runnning a script generated in PACKNOTES.
+by running a script generated in PACKNOTES.
 .PP
 You may then mail your kits via \fIkitsend\fR or post them with \fIkitpost\fR.
 .SH OPTIONS
diff --git a/kit/manifake.man b/kit/manifake.man
index bb39d75..f5df926 100644
--- a/kit/manifake.man
+++ b/kit/manifake.man
@@ -1,20 +1,20 @@
-''' $Id: manifake.man 3 2006-08-25 21:39:07Z rmanfredi $
-'''
-'''  Copyright (c) 1991-1997, 2004-2006, Raphael Manfredi
-'''  
-'''  You may redistribute only under the terms of the Artistic Licence,
-'''  as specified in the README file that comes with the distribution.
-'''  You may reuse parts of this distribution only within the terms of
-'''  that same Artistic Licence; a copy of which may be found at the root
-'''  of the source tree for dist 4.0.
-'''
-''' $Log: manifake.man,v $
-''' Revision 3.0.1.1  1995/05/12  11:58:21  ram
-''' patch54: updated my e-mail address
-'''
-''' Revision 3.0  1993/08/18  12:04:33  ram
-''' Baseline for dist 3.0 netwide release.
-'''
+.\" $Id: manifake.man 3 2006-08-25 21:39:07Z rmanfredi $
+.\"
+.\"  Copyright (c) 1991-1997, 2004-2006, Raphael Manfredi
+.\"  
+.\"  You may redistribute only under the terms of the Artistic Licence,
+.\"  as specified in the README file that comes with the distribution.
+.\"  You may reuse parts of this distribution only within the terms of
+.\"  that same Artistic Licence; a copy of which may be found at the root
+.\"  of the source tree for dist 4.0.
+.\"
+.\" $Log: manifake.man,v $
+.\" Revision 3.0.1.1  1995/05/12  11:58:21  ram
+.\" patch54: updated my e-mail address
+.\"
+.\" Revision 3.0  1993/08/18  12:04:33  ram
+.\" Baseline for dist 3.0 netwide release.
+.\"
 .TH MANIFAKE 1 ram
 .SH NAME
 manifake \- creates a MANIFEST.new out of a MANIFEST file
diff --git a/mcon/Jmakefile b/mcon/Jmakefile
index b5579cd..7eafa32 100644
--- a/mcon/Jmakefile
+++ b/mcon/Jmakefile
@@ -20,7 +20,7 @@
 ;# patch16: added dependency generation stage
 ;#
 ;# Revision 3.0.1.2  1993/11/02  10:40:01  ram
-;# patch14: now invokes perload explicitely via perl executable
+;# patch14: now invokes perload explicitly via perl executable
 ;#
 ;# Revision 3.0.1.1  1993/08/25  08:46:44  ram
 ;# patch6: split unit installation to avoid shell command line overflow
diff --git a/mcon/Makefile.SH b/mcon/Makefile.SH
index 54a10b6..64b62d5 100755
--- a/mcon/Makefile.SH
+++ b/mcon/Makefile.SH
@@ -81,7 +81,7 @@ $spitshell >>Makefile <<'!NO!SUBS!'
 # patch16: added dependency generation stage
 #
 # Revision 3.0.1.2  1993/11/02  10:40:01  ram
-# patch14: now invokes perload explicitely via perl executable
+# patch14: now invokes perload explicitly via perl executable
 #
 # Revision 3.0.1.1  1993/08/25  08:46:44  ram
 # patch6: split unit installation to avoid shell command line overflow
diff --git a/mcon/NOTES b/mcon/NOTES
index b6762e0..542910f 100644
--- a/mcon/NOTES
+++ b/mcon/NOTES
@@ -144,7 +144,7 @@ For instance, d_const.U writes:
 	?C:HASCONST ~ %<:
 
 so that the HASCONST hype is loaded iff the unit (%<) is wanted. This is why
-the ?H: lines are also explicitely tied to the wanted-ness of the d_const
+the ?H: lines are also explicitly tied to the wanted-ness of the d_const
 symbol, by writing:
 
 	?H:?%<:#$d_const HASCONST	/**/
diff --git a/mcon/U/Configdir.U b/mcon/U/Configdir.U
index f9a6c00..6833542 100644
--- a/mcon/U/Configdir.U
+++ b/mcon/U/Configdir.U
@@ -10,7 +10,7 @@
 ?RCS:
 ?RCS: $Log: Configdir.U,v $
 ?RCS: Revision 3.0.1.1  1997/02/28  14:58:36  ram
-?RCS: patch61: have README explicitely mention the package name
+?RCS: patch61: have README explicitly mention the package name
 ?RCS:
 ?RCS: Revision 3.0  1993/08/18  12:04:49  ram
 ?RCS: Baseline for dist 3.0 netwide release.
diff --git a/mcon/U/Finish.U b/mcon/U/Finish.U
index da3da8e..2848f2b 100644
--- a/mcon/U/Finish.U
+++ b/mcon/U/Finish.U
@@ -74,7 +74,7 @@ esac
 
 ?X:
 ?X: Turn silent mode off from now on (we want a verbose file extraction).
-?X: This means we have to explicitely test for '$silent' from now on to
+?X: This means we have to explicitly test for '$silent' from now on to
 ?X: strip off any verbose messages.
 ?X:
 echo " "
diff --git a/mcon/U/Guess.U b/mcon/U/Guess.U
index 6f6e6b4..4080a17 100644
--- a/mcon/U/Guess.U
+++ b/mcon/U/Guess.U
@@ -14,7 +14,7 @@
 ?RCS:
 ?RCS: Revision 3.0.1.4  1994/10/29  15:53:55  ram
 ?RCS: patch36: added ?F: line for metalint file checking
-?RCS: patch36: call ./xenix explicitely instead of relying on PATH
+?RCS: patch36: call ./xenix explicitly instead of relying on PATH
 ?RCS:
 ?RCS: Revision 3.0.1.3  1993/12/15  08:14:35  ram
 ?RCS: patch15: variable d_bsd was not always set properly
diff --git a/mcon/U/Head.U b/mcon/U/Head.U
index b0acc81..568597b 100644
--- a/mcon/U/Head.U
+++ b/mcon/U/Head.U
@@ -234,7 +234,7 @@ true/[a-z]*)
 	;;
 esac
 ?X: Warn them if they use ksh on other systems, which are those where
-?X: we don't need ksh nor want to avoid it explicitely, yet are using it.
+?X: we don't need ksh nor want to avoid it explicitly, yet are using it.
 case "$inksh/$needksh-$avoidksh-" in
 true/--)
 		cat <<EOM
diff --git a/mcon/U/Inhdr.U b/mcon/U/Inhdr.U
index 3891138..2c1f176 100644
--- a/mcon/U/Inhdr.U
+++ b/mcon/U/Inhdr.U
@@ -13,7 +13,7 @@
 ?RCS: patch54: deleted tabs that caused some /bin/sh to core dump (ADO)
 ?RCS:
 ?RCS: Revision 3.0.1.1  1994/10/29  15:55:01  ram
-?RCS: patch36: call ./whoa explicitely instead of relying on PATH
+?RCS: patch36: call ./whoa explicitly instead of relying on PATH
 ?RCS:
 ?RCS: Revision 3.0  1993/08/18  12:05:01  ram
 ?RCS: Baseline for dist 3.0 netwide release.
diff --git a/mcon/U/Inlibc.U b/mcon/U/Inlibc.U
index 802ff57..2a7fea3 100644
--- a/mcon/U/Inlibc.U
+++ b/mcon/U/Inlibc.U
@@ -10,7 +10,7 @@
 ?RCS:
 ?RCS: $Log: Inlibc.U,v $
 ?RCS: Revision 3.0.1.1  1994/10/29  15:55:08  ram
-?RCS: patch36: call ./whoa explicitely instead of relying on PATH
+?RCS: patch36: call ./whoa explicitly instead of relying on PATH
 ?RCS:
 ?RCS: Revision 3.0  1993/08/18  12:05:03  ram
 ?RCS: Baseline for dist 3.0 netwide release.
diff --git a/mcon/U/Setvar.U b/mcon/U/Setvar.U
index 05cf856..924c930 100644
--- a/mcon/U/Setvar.U
+++ b/mcon/U/Setvar.U
@@ -10,7 +10,7 @@
 ?RCS:
 ?RCS: $Log: Setvar.U,v $
 ?RCS: Revision 3.0.1.1  1994/10/29  15:58:28  ram
-?RCS: patch36: call ./whoa explicitely instead of relying on PATH
+?RCS: patch36: call ./whoa explicitly instead of relying on PATH
 ?RCS:
 ?RCS: Revision 3.0  1993/08/18  12:05:16  ram
 ?RCS: Baseline for dist 3.0 netwide release.
diff --git a/mcon/U/Warn_v7EXT.U b/mcon/U/Warn_v7EXT.U
index b669172..4a451ba 100644
--- a/mcon/U/Warn_v7EXT.U
+++ b/mcon/U/Warn_v7EXT.U
@@ -10,7 +10,7 @@
 ?RCS:
 ?RCS: $Log: Warn_v7EXT.U,v $
 ?RCS: Revision 3.0.1.1  1994/10/29  16:01:42  ram
-?RCS: patch36: call ./v7 explicitely instead of relying on PATH
+?RCS: patch36: call ./v7 explicitly instead of relying on PATH
 ?RCS:
 ?RCS: Revision 3.0  1993/08/18  12:05:17  ram
 ?RCS: Baseline for dist 3.0 netwide release.
diff --git a/mcon/U/Warn_v7ND.U b/mcon/U/Warn_v7ND.U
index eed3edc..d3255fe 100644
--- a/mcon/U/Warn_v7ND.U
+++ b/mcon/U/Warn_v7ND.U
@@ -10,7 +10,7 @@
 ?RCS:
 ?RCS: $Log: Warn_v7ND.U,v $
 ?RCS: Revision 3.0.1.1  1994/10/29  16:01:46  ram
-?RCS: patch36: call ./v7 explicitely instead of relying on PATH
+?RCS: patch36: call ./v7 explicitly instead of relying on PATH
 ?RCS:
 ?RCS: Revision 3.0  1993/08/18  12:05:18  ram
 ?RCS: Baseline for dist 3.0 netwide release.
diff --git a/mcon/U/abortsig.U b/mcon/U/abortsig.U
index 392e28e..37af2ae 100644
--- a/mcon/U/abortsig.U
+++ b/mcon/U/abortsig.U
@@ -10,7 +10,7 @@
 ?RCS:
 ?RCS: $Log: abortsig.U,v $
 ?RCS: Revision 3.0.1.1  1994/10/29  16:02:19  ram
-?RCS: patch36: call ./usg explicitely instead of relying on PATH
+?RCS: patch36: call ./usg explicitly instead of relying on PATH
 ?RCS:
 ?RCS: Revision 3.0  1993/08/18  12:05:20  ram
 ?RCS: Baseline for dist 3.0 netwide release.
diff --git a/mcon/U/afs.U b/mcon/U/afs.U
index c1cf0b7..d1e6dc5 100644
--- a/mcon/U/afs.U
+++ b/mcon/U/afs.U
@@ -10,7 +10,7 @@
 ?RCS: 
 ?RCS: $Log: afs.U,v $
 ?RCS: Revision 3.0.1.2  1997/02/28  15:23:06  ram
-?RCS: patch61: can now explicitely tell Configure whether AFS is running
+?RCS: patch61: can now explicitly tell Configure whether AFS is running
 ?RCS:
 ?RCS: Revision 3.0.1.1  1993/08/24  12:24:43  ram
 ?RCS: patch3: created
diff --git a/mcon/U/bin.U b/mcon/U/bin.U
index da5212f..60f8992 100644
--- a/mcon/U/bin.U
+++ b/mcon/U/bin.U
@@ -39,7 +39,7 @@
 ?S:.
 ?S:installbin:
 ?S:	This variable is the same as binexp unless AFS is running in which case
-?S:	the user is explicitely prompted for it. This variable should always
+?S:	the user is explicitly prompted for it. This variable should always
 ?S:	be used in your makefiles for maximum portability.
 ?S:.
 ?C:BIN:
diff --git a/mcon/U/ccflags.U b/mcon/U/ccflags.U
index 4cbb80a..93a01c9 100644
--- a/mcon/U/ccflags.U
+++ b/mcon/U/ccflags.U
@@ -33,7 +33,7 @@
 ?RCS: patch23: magically added cc flags now only done the first time
 ?RCS:
 ?RCS: Revision 3.0.1.3  1993/09/13  15:58:29  ram
-?RCS: patch10: explicitely mention -DDEBUG just in case they need it (WAD)
+?RCS: patch10: explicitly mention -DDEBUG just in case they need it (WAD)
 ?RCS: patch10: removed all the "tans" variable usage (WAD)
 ?RCS:
 ?RCS: Revision 3.0.1.2  1993/08/27  14:39:38  ram
diff --git a/mcon/U/d_bsdjmp.U b/mcon/U/d_bsdjmp.U
index ec11cf9..071ad35 100644
--- a/mcon/U/d_bsdjmp.U
+++ b/mcon/U/d_bsdjmp.U
@@ -13,7 +13,7 @@
 ?RCS: patch61: added ?F: metalint hint
 ?RCS:
 ?RCS: Revision 3.0.1.1  1994/10/29  16:09:09  ram
-?RCS: patch36: call ./usg explicitely instead of relying on PATH
+?RCS: patch36: call ./usg explicitly instead of relying on PATH
 ?RCS:
 ?RCS: Revision 3.0  1993/08/18  12:05:45  ram
 ?RCS: Baseline for dist 3.0 netwide release.
diff --git a/mcon/U/d_eofpipe.U b/mcon/U/d_eofpipe.U
index ab75282..f17e2ae 100644
--- a/mcon/U/d_eofpipe.U
+++ b/mcon/U/d_eofpipe.U
@@ -10,7 +10,7 @@
 ?RCS:
 ?RCS: $Log: d_eofpipe.U,v $
 ?RCS: Revision 3.0.1.1  1994/10/29  16:12:40  ram
-?RCS: patch36: call ./bsd explicitely instead of relying on PATH
+?RCS: patch36: call ./bsd explicitly instead of relying on PATH
 ?RCS:
 ?RCS: Revision 3.0  1993/08/18  12:05:57  ram
 ?RCS: Baseline for dist 3.0 netwide release.
diff --git a/mcon/U/d_gethname.U b/mcon/U/d_gethname.U
index a44da38..e9e4560 100644
--- a/mcon/U/d_gethname.U
+++ b/mcon/U/d_gethname.U
@@ -10,7 +10,7 @@
 ?RCS:
 ?RCS: $Log: d_gethname.U,v $
 ?RCS: Revision 3.0.1.1  1994/10/29  16:13:00  ram
-?RCS: patch36: call ./xenix explicitely instead of relying on PATH
+?RCS: patch36: call ./xenix explicitly instead of relying on PATH
 ?RCS:
 ?RCS: Revision 3.0  1993/08/18  12:06:11  ram
 ?RCS: Baseline for dist 3.0 netwide release.
diff --git a/mcon/U/d_keepsig.U b/mcon/U/d_keepsig.U
index e2a1d6a..f99016b 100644
--- a/mcon/U/d_keepsig.U
+++ b/mcon/U/d_keepsig.U
@@ -16,7 +16,7 @@
 ?RCS: patch45: protected "sh -c" within backquotes for Linux and SGI
 ?RCS:
 ?RCS: Revision 3.0.1.2  1994/10/29  16:13:59  ram
-?RCS: patch36: call ./bsd explicitely instead of relying on PATH
+?RCS: patch36: call ./bsd explicitly instead of relying on PATH
 ?RCS:
 ?RCS: Revision 3.0.1.1  1993/10/16  13:48:47  ram
 ?RCS: patch12: comment for SIGNALS_KEPT was the other way round
diff --git a/mcon/U/d_newsadm.U b/mcon/U/d_newsadm.U
index ca454de..23eb1a1 100644
--- a/mcon/U/d_newsadm.U
+++ b/mcon/U/d_newsadm.U
@@ -13,7 +13,7 @@
 ?RCS: patch61: added Guess dependency
 ?RCS:
 ?RCS: Revision 3.0.1.1  1994/10/29  16:14:33  ram
-?RCS: patch36: call ./eunice explicitely instead of relying on PATH
+?RCS: patch36: call ./eunice explicitly instead of relying on PATH
 ?RCS:
 ?RCS: Revision 3.0  1993/08/18  12:06:42  ram
 ?RCS: Baseline for dist 3.0 netwide release.
diff --git a/mcon/U/d_normsig.U b/mcon/U/d_normsig.U
index 2d5f426..1b0a8ba 100644
--- a/mcon/U/d_normsig.U
+++ b/mcon/U/d_normsig.U
@@ -13,7 +13,7 @@
 ?RCS: patch61: replaced .a with $_a all over the place
 ?RCS:
 ?RCS: Revision 3.0.1.1  1994/10/29  16:14:48  ram
-?RCS: patch36: call ./bsd explicitely instead of relying on PATH
+?RCS: patch36: call ./bsd explicitly instead of relying on PATH
 ?RCS:
 ?RCS: Revision 3.0  1993/08/18  12:06:44  ram
 ?RCS: Baseline for dist 3.0 netwide release.
diff --git a/mcon/U/d_setpgrp.U b/mcon/U/d_setpgrp.U
index a1dbfd6..b11d2d6 100644
--- a/mcon/U/d_setpgrp.U
+++ b/mcon/U/d_setpgrp.U
@@ -19,7 +19,7 @@
 ?RCS:
 ?RCS: Revision 3.0.1.1  1994/10/29  16:15:37  ram
 ?RCS: patch36: added 'ldflags' to the test compile line (ADO)
-?RCS: patch36: call ./usg explicitely instead of relying on PATH
+?RCS: patch36: call ./usg explicitly instead of relying on PATH
 ?RCS:
 ?RCS: Revision 3.0  1993/08/18  12:07:09  ram
 ?RCS: Baseline for dist 3.0 netwide release.
diff --git a/mcon/U/d_speedopt.U b/mcon/U/d_speedopt.U
index af57b99..07e68e8 100644
--- a/mcon/U/d_speedopt.U
+++ b/mcon/U/d_speedopt.U
@@ -10,7 +10,7 @@
 ?RCS:
 ?RCS: $Log: d_speedopt.U,v $
 ?RCS: Revision 3.0.1.1  1994/10/29  16:15:51  ram
-?RCS: patch36: call ./Cppsym explicitely instead of relying on PATH
+?RCS: patch36: call ./Cppsym explicitly instead of relying on PATH
 ?RCS:
 ?RCS: Revision 3.0  1993/08/18  12:07:27  ram
 ?RCS: Baseline for dist 3.0 netwide release.
diff --git a/mcon/U/d_tzmin.U b/mcon/U/d_tzmin.U
index f268a1b..b738f9e 100644
--- a/mcon/U/d_tzmin.U
+++ b/mcon/U/d_tzmin.U
@@ -10,7 +10,7 @@
 ?RCS:
 ?RCS: $Log: d_tzmin.U,v $
 ?RCS: Revision 3.0.1.1  1994/10/29  16:16:58  ram
-?RCS: patch36: call ./bsd explicitely instead of relying on PATH
+?RCS: patch36: call ./bsd explicitly instead of relying on PATH
 ?RCS:
 ?RCS: Revision 3.0  1993/08/18  12:07:49  ram
 ?RCS: Baseline for dist 3.0 netwide release.
diff --git a/mcon/U/d_vfork.U b/mcon/U/d_vfork.U
index 96410e7..fd59147 100644
--- a/mcon/U/d_vfork.U
+++ b/mcon/U/d_vfork.U
@@ -19,7 +19,7 @@
 ?RCS: patch27: modified to avoid spurious Whoa warnings (ADO)
 ?RCS:
 ?RCS: Revision 3.0.1.3  1994/05/06  14:59:09  ram
-?RCS: patch23: now explicitely ask whether vfork() should be used (ADO)
+?RCS: patch23: now explicitly ask whether vfork() should be used (ADO)
 ?RCS:
 ?RCS: Revision 3.0.1.2  1993/10/16  13:49:39  ram
 ?RCS: patch12: added magic for vfork()
@@ -40,7 +40,7 @@
 ?S:usevfork:
 ?S:	This variable is set to true when the user accepts to use vfork.
 ?S:	It is set to false when no vfork is available or when the user
-?S:	explicitely requests not to use vfork.
+?S:	explicitly requests not to use vfork.
 ?S:.
 ?C:HAS_VFORK (VFORK):
 ?C:	This symbol, if defined, indicates that vfork() exists.
diff --git a/mcon/U/i_termio.U b/mcon/U/i_termio.U
index c53cc1e..4d43bd7 100644
--- a/mcon/U/i_termio.U
+++ b/mcon/U/i_termio.U
@@ -10,7 +10,7 @@
 ?RCS:
 ?RCS: $Log: i_termio.U,v $
 ?RCS: Revision 3.0.1.4  1994/10/29  16:20:54  ram
-?RCS: patch36: call ./usg and ./Cppsym explicitely instead of relying on PATH
+?RCS: patch36: call ./usg and ./Cppsym explicitly instead of relying on PATH
 ?RCS:
 ?RCS: Revision 3.0.1.3  1994/08/29  16:26:38  ram
 ?RCS: patch32: don't include all three I_* symbols in config.h
diff --git a/mcon/U/myhostname.U b/mcon/U/myhostname.U
index fd6115c..45bb671 100644
--- a/mcon/U/myhostname.U
+++ b/mcon/U/myhostname.U
@@ -13,7 +13,7 @@
 ?RCS: patch61: improved hostname lookup by using ypmatch when NIS is used
 ?RCS:
 ?RCS: Revision 3.0.1.2  1994/10/29  16:25:43  ram
-?RCS: patch36: call ./xenix explicitely instead of relying on PATH
+?RCS: patch36: call ./xenix explicitly instead of relying on PATH
 ?RCS: patch36: now uses new Tr unit to convert to/from lowercase
 ?RCS:
 ?RCS: Revision 3.0.1.1  1994/06/20  07:06:20  ram
diff --git a/mcon/U/nametype.U b/mcon/U/nametype.U
index 72e036d..c00c5f7 100644
--- a/mcon/U/nametype.U
+++ b/mcon/U/nametype.U
@@ -10,7 +10,7 @@
 ?RCS:
 ?RCS: $Log: nametype.U,v $
 ?RCS: Revision 3.0.1.3  1994/10/29  16:26:09  ram
-?RCS: patch36: call ./usg and ./bsd explicitely instead of relying on PATH
+?RCS: patch36: call ./usg and ./bsd explicitly instead of relying on PATH
 ?RCS:
 ?RCS: Revision 3.0.1.2  1994/08/29  16:36:48  ram
 ?RCS: patch32: fixed typo: /etc/password -> /etc/passwd (WED)
diff --git a/mcon/U/nis.U b/mcon/U/nis.U
index 3bbcab9..7ed806e 100644
--- a/mcon/U/nis.U
+++ b/mcon/U/nis.U
@@ -165,4 +165,16 @@ case "$passcat" in
 '') passcat=':'
 	$test -f /etc/passwd && passcat='cat /etc/passwd';;
 esac
+case "$hostcat" in
+'') hostcat='cat /etc/hosts';;
+*)  ;;
+esac
+case "$groupcat" in
+'') groupcat='cat /etc/group';;
+*)  ;;
+esac
+case "$passcat" in
+'') passcat='cat /etc/passwd';;
+*)  ;;
+esac
 
diff --git a/mcon/U/registers.U b/mcon/U/registers.U
index 29a1d5d..c3197bd 100644
--- a/mcon/U/registers.U
+++ b/mcon/U/registers.U
@@ -13,7 +13,7 @@
 ?RCS: patch61: removed empty ?LINT lines
 ?RCS:
 ?RCS: Revision 3.0.1.1  1994/10/29  16:28:33  ram
-?RCS: patch36: call ./Cppsym explicitely instead of relying on PATH
+?RCS: patch36: call ./Cppsym explicitly instead of relying on PATH
 ?RCS:
 ?RCS: Revision 3.0  1993/08/18  12:09:41  ram
 ?RCS: Baseline for dist 3.0 netwide release.
diff --git a/mcon/U/sbrksmart.U b/mcon/U/sbrksmart.U
index 40ed078..5484af1 100644
--- a/mcon/U/sbrksmart.U
+++ b/mcon/U/sbrksmart.U
@@ -10,7 +10,7 @@
 ?RCS:
 ?RCS: $Log: sbrksmart.U,v $
 ?RCS: Revision 3.0.1.2  1995/01/11  15:35:41  ram
-?RCS: patch45: now sets sbrksmart to undef explicitely when lacking sbrk()
+?RCS: patch45: now sets sbrksmart to undef explicitly when lacking sbrk()
 ?RCS: patch45: forgot a cast when using return value from sbrk()
 ?RCS:
 ?RCS: Revision 3.0.1.1  1994/01/24  14:16:45  ram
diff --git a/mcon/U/shm_for.U b/mcon/U/shm_for.U
index 5461257..c2c7c3a 100644
--- a/mcon/U/shm_for.U
+++ b/mcon/U/shm_for.U
@@ -10,7 +10,7 @@
 ?RCS:
 ?RCS: $Log: shm_for.U,v $
 ?RCS: Revision 3.0.1.1  1994/10/29  16:28:37  ram
-?RCS: patch36: call ./Cppsym explicitely instead of relying on PATH
+?RCS: patch36: call ./Cppsym explicitly instead of relying on PATH
 ?RCS:
 ?RCS: Revision 3.0  1993/08/18  12:09:46  ram
 ?RCS: Baseline for dist 3.0 netwide release.
diff --git a/mcon/U/sysman.U b/mcon/U/sysman.U
index 5c38d0b..8a010ad 100644
--- a/mcon/U/sysman.U
+++ b/mcon/U/sysman.U
@@ -10,7 +10,7 @@
 ?RCS:
 ?RCS: $Log: sysman.U,v $
 ?RCS: Revision 3.0.1.2  1994/06/20  07:08:43  ram
-?RCS: patch30: now explicitely states that /usr/man/man1 is the default
+?RCS: patch30: now explicitly states that /usr/man/man1 is the default
 ?RCS: patch30: added /usr/local/man/man1 to the search list
 ?RCS:
 ?RCS: Revision 3.0.1.1  1993/09/13  16:13:50  ram
diff --git a/mcon/U/uidtype.U b/mcon/U/uidtype.U
index f34e581..e5d60ba 100644
--- a/mcon/U/uidtype.U
+++ b/mcon/U/uidtype.U
@@ -17,7 +17,7 @@
 ?RCS:
 ?RCS: Revision 3.0.1.1  1994/05/13  15:28:27  ram
 ?RCS: patch27: made conform to its gidtype.U companion
-?RCS: patch27: question now explicitely mentions getuid()
+?RCS: patch27: question now explicitly mentions getuid()
 ?RCS:
 ?RCS: Revision 3.0  1993/08/18  12:09:56  ram
 ?RCS: Baseline for dist 3.0 netwide release.
diff --git a/mcon/U/voidflags.U b/mcon/U/voidflags.U
index e970bfc..5bcf0aa 100644
--- a/mcon/U/voidflags.U
+++ b/mcon/U/voidflags.U
@@ -29,7 +29,7 @@
 ?S:defvoidused:
 ?S:	This variable contains the default value of the VOIDUSED symbol (15).
 ?S:.
-?X:	Exceptionally, we have to explicitely alias the symbol name for
+?X:	Exceptionally, we have to explicitly alias the symbol name for
 ?X: config_h.SH, otherwise the comment would not appear.
 ?C:VOIDFLAGS ~ %<:
 ?C:	This symbol indicates how much support of the void type is given by this
diff --git a/mcon/U/yacc.U b/mcon/U/yacc.U
index 63a4678..60b466d 100644
--- a/mcon/U/yacc.U
+++ b/mcon/U/yacc.U
@@ -74,7 +74,7 @@ esac
 $cat <<EOH
 
 Your yacc program may need extra flags to normally process the parser sources.
-Do NOT specify any -d or -v flags here, since those are explicitely known
+Do NOT specify any -d or -v flags here, since those are explicitly known
 by the various Makefiles. However, if your machine has strange/undocumented
 options (like -Sr# on SCO to specify the maximum number of grammar rules), then
 please add them here.  To use no flags, specify the word "none".
diff --git a/mcon/man/mconfig.SH b/mcon/man/mconfig.SH
index 12ca4f9..7013904 100755
--- a/mcon/man/mconfig.SH
+++ b/mcon/man/mconfig.SH
@@ -18,88 +18,88 @@ echo "Extracting mcon/man/metaconfig.$manext (with variable substitutions)"
 $rm -f metaconfig.$manext
 $spitshell >metaconfig.$manext <<!GROK!THIS!
 .TH METACONFIG $manext "Version $VERSION PL$PATCHLEVEL"
-''' @(#) Manual page for metaconfig
-'''
-''' $Id: mconfig.SH 25 2008-05-28 11:19:25Z rmanfredi $
-'''
-'''  Copyright (c) 1991-1997, 2004-2006, Raphael Manfredi
-'''  
-'''  You may redistribute only under the terms of the Artistic Licence,
-'''  as specified in the README file that comes with the distribution.
-'''  You may reuse parts of this distribution only within the terms of
-'''  that same Artistic Licence; a copy of which may be found at the root
-'''  of the source tree for dist 4.0.
-'''
-''' $Log: mconfig.SH,v $
-''' Revision 3.0.1.17  1997/02/28  16:29:31  ram
-''' patch61: documents the runnning environment and the src.U unit
-''' patch61: added warnings for $_a and $_o, as well as "startsh"
-'''
-''' Revision 3.0.1.16  1995/09/25  09:17:48  ram
-''' patch59: documented new ?Y: directive
-''' patch59: fixed my e-mail address
-'''
-''' Revision 3.0.1.15  1995/07/25  14:18:24  ram
-''' patch56: added extra nroff escapes at line heads to keep leading dots
-''' patch56: documented new -G option
-''' patch56: extended description of the Csym.U unit
-''' patch56: now mentions what a compile-link test line should look like
-'''
-''' Revision 3.0.1.14  1995/05/12  12:25:02  ram
-''' patch54: documented new -K switch for knowledgeable users
-'''
-''' Revision 3.0.1.13  1995/01/30  14:46:39  ram
-''' patch49: documented new special units Prefixit.U and Prefixup.U
-'''
-''' Revision 3.0.1.12  1995/01/11  15:39:16  ram
-''' patch45: documents new -O option and new Getfile escape supports
-''' patch45: documents the & escape in Myread and the new cc symbol lookup
-'''
-''' Revision 3.0.1.11  1994/10/29  16:32:38  ram
-''' patch36: added nroff protection for lines beginning with '.'
-''' patch36: documents new ?F: line for file declarations
-''' patch36: added example showing how ./loc can be used
-'''
-''' Revision 3.0.1.10  1994/08/29  16:33:40  ram
-''' patch32: documented new Typedef.U unit for typedef lookup
-'''
-''' Revision 3.0.1.9  1994/06/20  07:10:14  ram
-''' patch30: added -L option for easier unit testing
-''' patch30: new -D and -U options supported by Configure
-'''
-''' Revision 3.0.1.8  1994/05/13  15:29:16  ram
-''' patch27: now understands macro definitions in ?H: lines
-'''
-''' Revision 3.0.1.7  1994/05/06  15:19:25  ram
-''' patch23: documented the new 'p' option in Getfile.U
-'''
-''' Revision 3.0.1.6  1994/01/24  14:19:47  ram
-''' patch16: symbols defined in a unit can be tagged "internal use only"
-''' patch16: documents new MailList.U special unit
-''' patch16: new general <\$variable> macro substitution
-'''
-''' Revision 3.0.1.5  1993/10/16  13:51:50  ram
-''' patch12: new option -M to activate ?M: lines
-''' patch12: documents new ?M: lines format
-''' patch12: new internal Makefile command cm_h_weed for ?M: support
-''' patch12: documents usage for new confmagic.h file
-'''
-''' Revision 3.0.1.4  1993/09/09  11:50:35  ram
-''' patch9: lots of typo fixes and abusive variable substitution problems
-'''
-''' Revision 3.0.1.3  1993/08/30  08:53:51  ram
-''' patch8: wrongly stated that patchlevel.h should not be part of MANIFEST.new
-'''
-''' Revision 3.0.1.2  1993/08/24  12:13:32  ram
-''' patch3: typo fixes
-'''
-''' Revision 3.0.1.1  1993/08/19  06:42:23  ram
-''' patch1: leading config.sh searching was not aborting properly
-'''
-''' Revision 3.0  1993/08/18  12:10:14  ram
-''' Baseline for dist 3.0 netwide release.
-'''
-'''
+.\" @(#) Manual page for metaconfig
+.\"
+.\" $Id: mconfig.SH 25 2008-05-28 11:19:25Z rmanfredi $
+.\"
+.\"  Copyright (c) 1991-1997, 2004-2006, Raphael Manfredi
+.\"
+.\"  You may redistribute only under the terms of the Artistic Licence,
+.\"  as specified in the README file that comes with the distribution.
+.\"  You may reuse parts of this distribution only within the terms of
+.\"  that same Artistic Licence; a copy of which may be found at the root
+.\"  of the source tree for dist 4.0.
+.\"
+.\" $Log: mconfig.SH,v $
+.\" Revision 3.0.1.17  1997/02/28  16:29:31  ram
+.\" patch61: documents the running environment and the src.U unit
+.\" patch61: added warnings for $_a and $_o, as well as "startsh"
+.\"
+.\" Revision 3.0.1.16  1995/09/25  09:17:48  ram
+.\" patch59: documented new ?Y: directive
+.\" patch59: fixed my e-mail address
+.\"
+.\" Revision 3.0.1.15  1995/07/25  14:18:24  ram
+.\" patch56: added extra nroff escapes at line heads to keep leading dots
+.\" patch56: documented new -G option
+.\" patch56: extended description of the Csym.U unit
+.\" patch56: now mentions what a compile-link test line should look like
+.\"
+.\" Revision 3.0.1.14  1995/05/12  12:25:02  ram
+.\" patch54: documented new -K switch for knowledgeable users
+.\"
+.\" Revision 3.0.1.13  1995/01/30  14:46:39  ram
+.\" patch49: documented new special units Prefixit.U and Prefixup.U
+.\"
+.\" Revision 3.0.1.12  1995/01/11  15:39:16  ram
+.\" patch45: documents new -O option and new Getfile escape supports
+.\" patch45: documents the & escape in Myread and the new cc symbol lookup
+.\"
+.\" Revision 3.0.1.11  1994/10/29  16:32:38  ram
+.\" patch36: added nroff protection for lines beginning with '.'
+.\" patch36: documents new ?F: line for file declarations
+.\" patch36: added example showing how ./loc can be used
+.\"
+.\" Revision 3.0.1.10  1994/08/29  16:33:40  ram
+.\" patch32: documented new Typedef.U unit for typedef lookup
+.\"
+.\" Revision 3.0.1.9  1994/06/20  07:10:14  ram
+.\" patch30: added -L option for easier unit testing
+.\" patch30: new -D and -U options supported by Configure
+.\"
+.\" Revision 3.0.1.8  1994/05/13  15:29:16  ram
+.\" patch27: now understands macro definitions in ?H: lines
+.\"
+.\" Revision 3.0.1.7  1994/05/06  15:19:25  ram
+.\" patch23: documented the new 'p' option in Getfile.U
+.\"
+.\" Revision 3.0.1.6  1994/01/24  14:19:47  ram
+.\" patch16: symbols defined in a unit can be tagged "internal use only"
+.\" patch16: documents new MailList.U special unit
+.\" patch16: new general <\$variable> macro substitution
+.\"
+.\" Revision 3.0.1.5  1993/10/16  13:51:50  ram
+.\" patch12: new option -M to activate ?M: lines
+.\" patch12: documents new ?M: lines format
+.\" patch12: new internal Makefile command cm_h_weed for ?M: support
+.\" patch12: documents usage for new confmagic.h file
+.\"
+.\" Revision 3.0.1.4  1993/09/09  11:50:35  ram
+.\" patch9: lots of typo fixes and abusive variable substitution problems
+.\"
+.\" Revision 3.0.1.3  1993/08/30  08:53:51  ram
+.\" patch8: wrongly stated that patchlevel.h should not be part of MANIFEST.new
+.\"
+.\" Revision 3.0.1.2  1993/08/24  12:13:32  ram
+.\" patch3: typo fixes
+.\"
+.\" Revision 3.0.1.1  1993/08/19  06:42:23  ram
+.\" patch1: leading config.sh searching was not aborting properly
+.\"
+.\" Revision 3.0  1993/08/18  12:10:14  ram
+.\" Baseline for dist 3.0 netwide release.
+.\"
+.\"
 .de Ex		\" Start of Example
 .sp
 .in +5
@@ -304,7 +304,7 @@ formats allowed:
 ?X: A contrived example
 ?X:
 ?MAKE:d_one two: three +four Five
-?MAKE:	-pick add \$@ %<
+?MAKE:	\-pick add \$@ %<
 ?Y:DEFAULT
 ?S:d_one:
 ?S:	First shell symbol, conditionally defines ONE.
@@ -365,10 +365,10 @@ of general interest.
 .PP
 Notes for writing .U files:
 .IP * 5
-Always use "rm -f" because there are systems where rm is interactive by
+Always use "rm \-f" because there are systems where rm is interactive by
 default.
 .IP *
-Do not use "set -- ..." because '--' does not work with every shell. Use
+Do not use "set \-\- ..." because '\-\-' does not work with every shell. Use
 "set x ...; shift".
 .IP *
 Do not use "unset ENV" since unset is not fully portable.  Say "ENV=''"
@@ -377,7 +377,7 @@ instead.
 Always use echo " " (with a space) because of Eunice systems.
 .IP *
 Only use \fBtest\fR with \-r, \-w, \-f or \-d since those are the only
-portable switches. In particular, avoid "test -x".
+portable switches. In particular, avoid "test \-x".
 .IP *
 Use only programs that came with V7, so that you know everyone has them.
 .IP *
@@ -417,7 +417,7 @@ Use the \fI\$_o\fR and \fI\$_a\fR variables instead (see Unix.U).
 .IP *
 When doing a compile-link-execute test, always write it like this:
 .Ex
-\$\&cc \$\&ccflags \$\&ldflags try.c -o try \$\&libs
+\$\&cc \$\&ccflags \$\&ldflags try.c \-o try \$\&libs
 .Ef
 because some systems require that linking flags be specified before
 the compiled target (with the exception of trailing linking libraries).
@@ -459,9 +459,9 @@ UU/*
 config.sh
 config.h
 .Ef
-'''
-''' O p t i o n s
-'''
+.\"
+.\" O p t i o n s
+.\"
 .SH OPTIONS
 The following options are recognized by \fImetaconfig\fR:
 .TP 15
@@ -514,7 +514,7 @@ not be lost when facing a new distribution.
 .TP
 \fB\-L\fI dir\fR
 Override default library location. Normally only useful for metaconfig
-maintainers to locally use the units being developped instead of the
+maintainers to locally use the units being developed instead of the
 publicly available ones. The \fIdir\fR specified is the one containing the
 units \fIU\fR directory.
 .TP
@@ -528,14 +528,14 @@ option permanently.
 .TP
 .B \-V
 Print version number and exit.
-'''
-''' T u t o r i a l
-'''
+.\"
+.\" T u t o r i a l
+.\"
 .SH TUTORIAL
 This (long) section is an introduction to \fImetaconfig\fR, in which we will
 learn all the basics. If you already know how to use \fImetaconfig\fR, you
 may safely skip to the next section.
-'''
+.\"
 .SS Overview
 .PP
 Usually when you want to get some source package to compile on a given
@@ -550,8 +550,8 @@ some standard trial, guessing and messing around with system includes and
 types.
 .PP
 However, despite all his talent, the author cannot possibly know that
-some system has a broken system call, or that some sytem structure lacks
-one otherwise standard field, or simply wheter a given include file exists
+some system has a broken system call, or that some system structure lacks
+one otherwise standard field, or simply whether a given include file exists
 or not. And I'm not considering the implicit assumptions, like the type
 returned by the \fImalloc()\fR function or the presence of the \fIrename()\fR
 system call to name a few. But that knowledge is necessary to achieve real
@@ -588,7 +588,7 @@ unit being responsible for defining a small number of shell and/or C
 symbols. Units are assembled together at the final stage, honoring
 the dependency graph (one unit may need the result of several other
 units which are then placed before in the script).
-'''
+.\"
 .SS Symbols
 .PP
 Symbols are the most important thing in the \fImetaconfig\fR world. They
@@ -613,7 +613,7 @@ paragraph could sound. In a C file, you need to include the Configure-produced
 \fIconfig.h\fR file, and you must wrap your shell script or Makefile in a .SH
 file and you may reference the shell symbol only in the variable
 substitution part of that .SH file. More on this later.
-'''
+.\"
 .SS Source Files
 .PP
 Symbols may only appear in a limited set of source files, because
@@ -631,7 +631,7 @@ substitutions apply. Actually, \fIconfig.h\fR is produced by running the
 \fImetaconfig\fR-produced \fIconfig_h.SH\fR file, again using variable
 substitution. So we're going to look at that a little more closely since
 this is the heart of the whole \fIconfiguration\fR scheme...
-'''
+.\"
 .SS Variable Substitution
 .PP
 There is shell construct called \fIhere document\fR which enables a
@@ -648,7 +648,7 @@ unless the identifier is surrounded by single quotes. For instance:
 .Ex
 var='first'
 tar='second'
-echo "--> first here document:"
+echo "\--> first here document:"
 cat <<EOM
 var='\$var'
 tar='\$\&tar'
@@ -672,7 +672,7 @@ echo \$\&tar
 .Ef
 The first here document has its content interpreted whilst the second
 one is output as-is. Both are useful in a .SH script, as we are about to see.
-'''
+.\"
 .SS Using .SH Scripts
 .PP
 A .SH script is usually produced by running the \fIMakeSH\fR script other
@@ -811,7 +811,7 @@ stored as a 64 bits quantity, \fIconfig.sh\fR will set \fIintsize\fR to
 On this machine, the int type is 8 bytes
 .Ef
 which is correct. Congratulations! We have just configured a shell script!!
-'''
+.\"
 .SS Producing config.h
 .PP
 We can now have a look at the way \fIconfig.h\fR is produced out of
@@ -965,7 +965,7 @@ configured C code, since \fImetaconfig\fR will know that you need
 those symbols and will generate a suitable \fIconfig_h.SH\fR file as
 well as all the necessary code in \fIConfigure\fR to compute a
 proper value for them (by assigning values to associated shell variables).
-'''
+.\"
 .SS Running Metaconfig
 .PP
 Let's focus on the \fImetaconfig\fR program for a while to understand how
@@ -1026,7 +1026,7 @@ required are removed and a second Makefile is generated. This time, we know
 about all the units and their respective orders, optional units having been
 removed and default values computed for their shell symbols. The \fIConfigure\fR
 script can then be generated, along with \fIconfig_h.SH\fR. We're done.
-'''
+.\"
 .SS Conventions
 .PP
 Proper conventions needs to be followed to make the whole process sound.
@@ -1056,7 +1056,7 @@ answer in \fI\$dflt\fR and places the user answer in \fI\$ans\fR. This is
 not documented in this manual page: you have to go and look at the unit
 itself to understand which variables are used and how the unit is to be
 used.
-'''
+.\"
 .SS Using The Glossary
 .PP
 The Glossary file is automatically produced by the \fImakegloss\fR script,
@@ -1071,7 +1071,7 @@ about \fImetaconfig\fR to do so quickly and efficiently. However, never
 forget to properly document your work in the ?S: and ?C: lines, or other
 people will not be able to reuse it. Remember about the time where you
 had only the Glossary and this manual page to get started.
-'''
+.\"
 .SS Conclusion
 .PP
 Now that you know the \fImetaconfig\fR basics, you should read the
@@ -1082,7 +1082,7 @@ commands you are allowed to use.
 .SH REFERENCE
 This section documents the internals of \fImetaconfig\fR, basically the
 unit syntax, the special units you should know about and the hint files.
-'''
+.\"
 .SS General Unit Syntax
 .PP
 A metaconfig unit is divided into two distinct parts. The header section
@@ -1166,7 +1166,7 @@ the C symbol will be controlled, i.e. if the \fIalias\fR symbol is wanted,
 then that C symbol will be written in the \fIconfig_h.SH\fR file. Usually,
 the alias is just '%<' (stands for the unit's name) and there is also
 a ?W: line mapping a C symbol to the \fIalias\fR. Also the relevant parts
-of the ?H: lines are explicitely protected by a '?%<' condition. See
+of the ?H: lines are explicitly protected by a '?%<' condition. See
 the symbol aliasing paragraph for more details.
 The remaining of the line is the optional \fIobsolete symbol list\fR,
 which lists old equivalents for the new \fIsymbol_name\fR.
@@ -1290,7 +1290,7 @@ hints that can be used.
 ?INIT:\fIinitialization code\fR
 The initialization code specified by this line will be loaded at the top
 of the \fIConfigure\fR script provided the unit is needed.
-'''
+.\"
 .SS C Symbol Aliasing
 .PP
 Sometimes it is not possible to rely on \fImetaconfig\fR's own default
@@ -1370,7 +1370,7 @@ It simply tries to compile a sample C program using the \fIconst\fR keyword.
 If it can, then it will define \fI\$\&d_const\fR via the \fI\$\&setvar\fR
 fonction (defined by the \fISetvar.U\fR unit). See the paragraph about
 special units for more details.
-'''
+.\"
 .SS Make Commands
 .PP
 On the ?MAKE: command line, you may write a shell command to be executed as-is
@@ -1435,7 +1435,7 @@ As a side note, \fImetaconfig\fR generates a \fI-cond\fR command internally
 to deal with conditional dependencies. You should not use it by yourself,
 but you will see it if scanning the generated \fIMakefile\fR in the \fI.MT\fR
 directory.
-'''
+.\"
 .SS Hardwired Macros
 .PP
 The following macros are recognized by the \fIwipe\fR command and subsituted
@@ -1478,7 +1478,7 @@ which is replaced at Configure-generation time by the value of \fIvariable\fR
 taken from your \fI.package\fR file. Eventually, the old hardwired macro
 format will disappear, and <\$baserev> will replace <BASEREV> in all the
 supplied units.
-'''
+.\"
 .SS Special Units
 .PP
 The following special units are used to factorize code and provide higher
@@ -1507,7 +1507,7 @@ check whether a given C symbol is defined or not. A typical use is:
 set symbol result [-fva] [previous]
 eval \$\&csym
 .Ef
-That will set the \fIresult\fR variable to 'true' if the 
+That will set the \fIresult\fR variable to 'true' if the
 function [-f],
 variable [-v] or
 array [-a]
@@ -1642,7 +1642,7 @@ argument specifies the file to be located, the second argument is what
 will be returned if the search fails, and the reamining arguments are a
 list of directories where the file is to be searched. For instance:
 .Ex
-dflt=\`./loc sendmail.cf X /usr/lib /var/adm/sendmail /lib\`
+dflt=\`./loc sendmail.cf X /usr/lib /var/lib/sendmail /lib\`
 .Ef
 would set \fI\$dflt\fR to \fIX\fR if no \fIsendmail.cf\fR file was found
 under the listed directories, or something like \fI/usr/lib/sendmail.cf\fR
@@ -1718,7 +1718,7 @@ to an empty string if the prefix changed, to \fI\$var\fR otherwise.
 .TP
 Prefixup.U
 The intent of thit unit is similar to that of Prefixit.U, i.e. it helps
-fixing the default string to accomodate prefix changes. However, the shell
+fixing the default string to accommodate prefix changes. However, the shell
 variable \fI\$prefixup\fR, when evaluated, will only restore ~name expansions,
 should prefix use such an escape mechanism. Use it as:
 .Ex
@@ -1768,7 +1768,7 @@ This unit produces the \fIwhoa\fR script, which emits a warning when the
 its old previous value held in \fI\$was\fR. Upon return, \fI\$td\fR and
 \fI\$tu\fR hold the proper value to \fIdefine\fR or \fIundef\fR the variable.
 See examples in \fIInlibc.U\fR.
-'''
+.\"
 .SS Builtin Pre-processor
 .PP
 Each unit to be included in \fIConfigure\fR is ran through a built-in
@@ -1868,7 +1868,7 @@ done
 .Ef
 and have an extra loop trying two successive values for the \fIs_timezone\fR
 variable, but only if needed.
-'''
+.\"
 .SS Obsolete Symbols
 .PP
 Obsolete symbols are preserved to ease the transition with older
@@ -1882,7 +1882,7 @@ The lifetime for obsolete symbols is one full revision, i.e. they will
 be removed when the next base revision is issued (patch upgrades do not
 count of course). Therefore, it is wise to translate your sources and
 start using the new symbols as soon as possible.
-'''
+.\"
 .SS Configure Hints
 .PP
 It may happen that the internal configuration logic makes the wrong choices.
@@ -1932,7 +1932,7 @@ Note that you don't have to provide \fIall\fR the hints known by
 possible choice. The heuristic tests ran to compute the possible hint
 candidates are flaky. If you have new values or different tests, please send
 them to me...
-'''
+.\"
 .SS Overriding Choices
 .PP
 If you create a \fIconfig.over\fR file in the top level directory,
@@ -1943,7 +1943,7 @@ you a chance to patch the values stored in there.
 This is distinct from the hints approach in that it is a local file, which
 the user is free to create for his own usage. You should not provide such
 a file yourself, but let the user know about this possibility.
-'''
+.\"
 .SS Configure Options
 .PP
 The \fIConfigure\fR script may be called with some options specified on the
@@ -2015,7 +2015,7 @@ possible to use '\fB-U\fI symbol\fR' which will set \fIsymbol\fR to 'undef'.
 Print the version number of the \fImetaconfig\fR that generated this
 .I Configure
 script and exit.
-'''
+.\"
 .SS Running Environment
 Upon starting, \fIConfigure\fR creates a local \fIUU\fR directory and runs
 from there. The directory is removed when Configure ends, but this means
@@ -2044,7 +2044,7 @@ When running \fIConfigure\fR remotely, the .SH files are extracted in the
 build directory, not in the source tree. However, it requires some kind of
 a \fImake\fR support to be able to compile things in a build directory whilst
 the sources lie elsewhere.
-'''
+.\"
 .SS Using Magic Redefinitions
 .PP
 By making use of the \fB\-M\fR switch, some magic remappings may take place
@@ -2064,7 +2064,7 @@ on \fImemcpy()\fR if no \fIbcopy()\fR is available locally, or transform
 bother about the \fIHAS_VFORK\fR symbol.
 .PP
 This configuration magic is documented in the Glossary file.
-'''
+.\"
 .SS Unit Templates
 .PP
 There is a set of unit templates in the \fImetaconfig\fR source directory,
diff --git a/mcon/man/mlint.SH b/mcon/man/mlint.SH
index c842f4f..62740b2 100755
--- a/mcon/man/mlint.SH
+++ b/mcon/man/mlint.SH
@@ -18,54 +18,54 @@ echo "Extracting mcon/man/metalint.$manext (with variable substitutions)"
 $rm -f metalint.$manext
 $spitshell >metalint.$manext <<!GROK!THIS!
 .TH METALINT $manext "Version $VERSION PL$PATCHLEVEL"
-''' @(#) Manual page for metalint
-'''
-''' $Id: mlint.SH 1 2006-08-24 12:32:52Z rmanfredi $
-'''
-'''  Copyright (c) 1991-1997, 2004-2006, Raphael Manfredi
-'''  
-'''  You may redistribute only under the terms of the Artistic Licence,
-'''  as specified in the README file that comes with the distribution.
-'''  You may reuse parts of this distribution only within the terms of
-'''  that same Artistic Licence; a copy of which may be found at the root
-'''  of the source tree for dist 4.0.
-'''
-''' $Log: mlint.SH,v $
-''' Revision 3.0.1.9  1997/02/28  16:30:25  ram
-''' patch61: new "create" and "empty" lint directives
-''' patch61: documented new messages
-'''
-''' Revision 3.0.1.8  1995/09/25  09:18:07  ram
-''' patch59: documented new ?Y: directive
-''' patch59: fixed my e-mail address
-'''
-''' Revision 3.0.1.7  1995/07/25  14:18:51  ram
-''' patch56: added two new warnings for : comments lines in Configure
-'''
-''' Revision 3.0.1.6  1994/10/29  16:33:56  ram
-''' patch36: documents new ?F: lines and the related metalint warnings
-''' patch36: removed statement in BUGS since all warnings may now be shut
-'''
-''' Revision 3.0.1.5  1994/05/06  15:20:30  ram
-''' patch23: added -L switch to override public unit repository path
-''' patch23: two new warnings concerning last unit lines
-'''
-''' Revision 3.0.1.4  1994/01/24  14:20:39  ram
-''' patch16: can now easily suppress warning about made "special units"
-'''
-''' Revision 3.0.1.3  1993/11/10  17:37:39  ram
-''' patch14: documents stale ?M: dependency check
-'''
-''' Revision 3.0.1.2  1993/10/16  13:52:23  ram
-''' patch12: added support for ?M: lines and fixed some typos
-'''
-''' Revision 3.0.1.1  1993/08/19  06:42:24  ram
-''' patch1: leading config.sh searching was not aborting properly
-'''
-''' Revision 3.0  1993/08/18  12:10:15  ram
-''' Baseline for dist 3.0 netwide release.
-'''
-'''
+.\" @(#) Manual page for metalint
+.\"
+.\" $Id: mlint.SH 1 2006-08-24 12:32:52Z rmanfredi $
+.\"
+.\"  Copyright (c) 1991-1997, 2004-2006, Raphael Manfredi
+.\"  
+.\"  You may redistribute only under the terms of the Artistic Licence,
+.\"  as specified in the README file that comes with the distribution.
+.\"  You may reuse parts of this distribution only within the terms of
+.\"  that same Artistic Licence; a copy of which may be found at the root
+.\"  of the source tree for dist 4.0.
+.\"
+.\" $Log: mlint.SH,v $
+.\" Revision 3.0.1.9  1997/02/28  16:30:25  ram
+.\" patch61: new "create" and "empty" lint directives
+.\" patch61: documented new messages
+.\"
+.\" Revision 3.0.1.8  1995/09/25  09:18:07  ram
+.\" patch59: documented new ?Y: directive
+.\" patch59: fixed my e-mail address
+.\"
+.\" Revision 3.0.1.7  1995/07/25  14:18:51  ram
+.\" patch56: added two new warnings for : comments lines in Configure
+.\"
+.\" Revision 3.0.1.6  1994/10/29  16:33:56  ram
+.\" patch36: documents new ?F: lines and the related metalint warnings
+.\" patch36: removed statement in BUGS since all warnings may now be shut
+.\"
+.\" Revision 3.0.1.5  1994/05/06  15:20:30  ram
+.\" patch23: added -L switch to override public unit repository path
+.\" patch23: two new warnings concerning last unit lines
+.\"
+.\" Revision 3.0.1.4  1994/01/24  14:20:39  ram
+.\" patch16: can now easily suppress warning about made "special units"
+.\"
+.\" Revision 3.0.1.3  1993/11/10  17:37:39  ram
+.\" patch14: documents stale ?M: dependency check
+.\"
+.\" Revision 3.0.1.2  1993/10/16  13:52:23  ram
+.\" patch12: added support for ?M: lines and fixed some typos
+.\"
+.\" Revision 3.0.1.1  1993/08/19  06:42:24  ram
+.\" patch1: leading config.sh searching was not aborting properly
+.\"
+.\" Revision 3.0  1993/08/18  12:10:15  ram
+.\" Baseline for dist 3.0 netwide release.
+.\"
+.\"
 .SH NAME
 metalint \- a metaconfig unit consistency checker
 .SH SYNOPSIS
@@ -104,7 +104,7 @@ Silent mode.
 .TP
 \fB\-L\fI dir\fR
 Override default library location. Normally only useful for metaconfig
-maintainers to locally check the units being developped instead of the
+maintainers to locally check the units being developed instead of the
 publicly available ones. The \fIdir\fR specified is the one containing the
 units \fIU\fR directory.
 .TP
@@ -219,7 +219,7 @@ The variable was already made visible as read-write by another unit.
 The variable was already made visible as read-only by another unit.
 .TP
 "(?W) variable '\$xxx' already wanted."
-Variable is already wanted, there is no need to explicitely ask for it here.
+Variable is already wanted, there is no need to explicitly ask for it here.
 .TP
 "(?W) variable '\$xxx' also locally defined."
 A wanted variable would conflict with a locally defined variable and could
@@ -400,7 +400,7 @@ dependency information. Use the \fIextern\fR lint hint to suppress this message.
 A file listed as a private UU file is not created by the special unit.
 Creation is detected by seing an explicit shell redirection to the file,
 not by an implicit creation (such as a C compilation would for instance).
-All special units should create only shell scripts explicitely anyway so this
+All special units should create only shell scripts explicitly anyway so this
 message cannot be suppressed via a lint hint.
 .TP
 "local file 'xxx' may override the one set by 'unit.U'."
diff --git a/mcon/man/mxref.SH b/mcon/man/mxref.SH
index 0b7ec6f..acbdf8b 100755
--- a/mcon/man/mxref.SH
+++ b/mcon/man/mxref.SH
@@ -18,35 +18,35 @@ echo "Extracting mcon/man/metaxref.$manext (with variable substitutions)"
 $rm -f metaxref.$manext
 $spitshell >metaxref.$manext <<!GROK!THIS!
 .TH METAXREF $manext "Version $VERSION PL$PATCHLEVEL"
-''' @(#) Manual page for metaxref
-'''
-''' $Id: mxref.SH 1 2006-08-24 12:32:52Z rmanfredi $
-'''
-'''  Copyright (c) 1991-1997, 2004-2006, Raphael Manfredi
-'''  
-'''  You may redistribute only under the terms of the Artistic Licence,
-'''  as specified in the README file that comes with the distribution.
-'''  You may reuse parts of this distribution only within the terms of
-'''  that same Artistic Licence; a copy of which may be found at the root
-'''  of the source tree for dist 4.0.
-'''
-''' $Log: mxref.SH,v $
-''' Revision 3.0.1.4  1997/02/28  16:30:39  ram
-''' patch61: new -L option to match metaconfig and metalint
-'''
-''' Revision 3.0.1.3  1994/10/29  16:34:26  ram
-''' patch36: the leading .TH was referring to metaconfig
-'''
-''' Revision 3.0.1.2  1993/10/16  13:52:46  ram
-''' patch12: added BUGS section
-'''
-''' Revision 3.0.1.1  1993/08/19  06:42:25  ram
-''' patch1: leading config.sh searching was not aborting properly
-'''
-''' Revision 3.0  1993/08/18  12:10:15  ram
-''' Baseline for dist 3.0 netwide release.
-'''
-'''
+.\" @(#) Manual page for metaxref
+.\"
+.\" $Id: mxref.SH 1 2006-08-24 12:32:52Z rmanfredi $
+.\"
+.\"  Copyright (c) 1991-1997, 2004-2006, Raphael Manfredi
+.\"  
+.\"  You may redistribute only under the terms of the Artistic Licence,
+.\"  as specified in the README file that comes with the distribution.
+.\"  You may reuse parts of this distribution only within the terms of
+.\"  that same Artistic Licence; a copy of which may be found at the root
+.\"  of the source tree for dist 4.0.
+.\"
+.\" $Log: mxref.SH,v $
+.\" Revision 3.0.1.4  1997/02/28  16:30:39  ram
+.\" patch61: new -L option to match metaconfig and metalint
+.\"
+.\" Revision 3.0.1.3  1994/10/29  16:34:26  ram
+.\" patch36: the leading .TH was referring to metaconfig
+.\"
+.\" Revision 3.0.1.2  1993/10/16  13:52:46  ram
+.\" patch12: added BUGS section
+.\"
+.\" Revision 3.0.1.1  1993/08/19  06:42:25  ram
+.\" patch1: leading config.sh searching was not aborting properly
+.\"
+.\" Revision 3.0  1993/08/18  12:10:15  ram
+.\" Baseline for dist 3.0 netwide release.
+.\"
+.\"
 .SH NAME
 metaxref \- builds cross-reference file/unit/item information
 .SH SYNOPSIS
diff --git a/pat/pat.man b/pat/pat.man
index bc957d1..68e37e2 100644
--- a/pat/pat.man
+++ b/pat/pat.man
@@ -1,42 +1,42 @@
 .rn '' }`
-''' $Id: pat.man 3 2006-08-25 21:39:07Z rmanfredi $
-'''
-'''  Copyright (c) 1991-1997, 2004-2006, Raphael Manfredi
-'''  
-'''  You may redistribute only under the terms of the Artistic Licence,
-'''  as specified in the README file that comes with the distribution.
-'''  You may reuse parts of this distribution only within the terms of
-'''  that same Artistic Licence; a copy of which may be found at the root
-'''  of the source tree for dist 4.0.
-'''
-''' $Log: pat.man,v $
-''' Revision 3.0.1.7  1997/02/28  16:32:45  ram
-''' patch61: documents contents of the message sent by patnotify
-'''
-''' Revision 3.0.1.6  1995/09/25  09:20:41  ram
-''' patch59: new -i option for patsend to add extra instructions
-'''
-''' Revision 3.0.1.5  1995/05/12  12:25:28  ram
-''' patch54: updated my e-mail address
-'''
-''' Revision 3.0.1.4  1994/10/29  16:38:31  ram
-''' patch36: documents new patlog script and the files it uses
-''' patch36: the RCS layer section has been extended slightly
-'''
-''' Revision 3.0.1.3  1993/08/27  14:40:19  ram
-''' patch7: random cleanup
-'''
-''' Revision 3.0.1.2  1993/08/25  14:05:02  ram
-''' patch6: new -q option for patsend and patnotify
-'''
-''' Revision 3.0.1.1  1993/08/24  12:15:42  ram
-''' patch3: added patnotify and patsnap
-''' patch3: patcol has a new -S option
-''' patch3: the users file built by mailagent is now listed under FILES
-'''
-''' Revision 3.0  1993/08/18  12:10:37  ram
-''' Baseline for dist 3.0 netwide release.
-'''
+.\" $Id: pat.man 3 2006-08-25 21:39:07Z rmanfredi $
+.\"
+.\"  Copyright (c) 1991-1997, 2004-2006, Raphael Manfredi
+.\"
+.\"  You may redistribute only under the terms of the Artistic Licence,
+.\"  as specified in the README file that comes with the distribution.
+.\"  You may reuse parts of this distribution only within the terms of
+.\"  that same Artistic Licence; a copy of which may be found at the root
+.\"  of the source tree for dist 4.0.
+.\"
+.\" $Log: pat.man,v $
+.\" Revision 3.0.1.7  1997/02/28  16:32:45  ram
+.\" patch61: documents contents of the message sent by patnotify
+.\"
+.\" Revision 3.0.1.6  1995/09/25  09:20:41  ram
+.\" patch59: new -i option for patsend to add extra instructions
+.\"
+.\" Revision 3.0.1.5  1995/05/12  12:25:28  ram
+.\" patch54: updated my e-mail address
+.\"
+.\" Revision 3.0.1.4  1994/10/29  16:38:31  ram
+.\" patch36: documents new patlog script and the files it uses
+.\" patch36: the RCS layer section has been extended slightly
+.\"
+.\" Revision 3.0.1.3  1993/08/27  14:40:19  ram
+.\" patch7: random cleanup
+.\"
+.\" Revision 3.0.1.2  1993/08/25  14:05:02  ram
+.\" patch6: new -q option for patsend and patnotify
+.\"
+.\" Revision 3.0.1.1  1993/08/24  12:15:42  ram
+.\" patch3: added patnotify and patsnap
+.\" patch3: patcol has a new -S option
+.\" patch3: the users file built by mailagent is now listed under FILES
+.\"
+.\" Revision 3.0  1993/08/18  12:10:37  ram
+.\" Baseline for dist 3.0 netwide release.
+.\"
 .de Sh
 .br
 .ne 5
@@ -48,10 +48,10 @@
 .if t .sp .5v
 .if n .sp
 ..
-'''
-'''     Set up \*(-- to give an unbreakable dash;
-'''     string Tr holds user defined translation string.
-'''
+.\"
+.\"     Set up \*(-- to give an unbreakable dash;
+.\"     string Tr holds user defined translation string.
+.\"
 .ie n \{\
 .tr \(*W-\*(Tr
 .ds -- \(*W-
@@ -229,7 +229,7 @@ then use that file list.
 If every file of the list is removed, \fIpat\fR will be aborted.
 .PP
 .I Patcil
-is used to do a ci -l on any listed files.
+is used to do a ci \-l on any listed files.
 (It is assumed that you always want to keep your files checked out.)
 In addition to the
 .B \-a
@@ -251,7 +251,7 @@ which is passed through to the ci to force unchanged files to be checked in.
 To check in a new trunk revision, I say
 .nf
 
-	patcil -s -f -a
+	patcil \-s \-f \-a
 
 .fi
 .PP
@@ -285,15 +285,15 @@ initialize the RCS file.
 can be used to reset the patch base to the current version when
 you've scrapped the previous patch sequence and are making a new distribution
 kit.
-What it really does is an rcs -Nlastpat:REV, where REV is the current
+What it really does is an rcs \-Nlastpat:REV, where REV is the current
 revision.
 If patdiff blows up and you want to set the patch base back to some previous
-version, you have to call rcs -Nlastpat:REV yourself.
+version, you have to call rcs \-Nlastpat:REV yourself.
 .PP
 .I Patdiff
 actually does the diffs that go into the patch, comparing whatever
-version -Nlastpat points to with the most recently checked in version.
-It then updates -Nlastpat to point to the current version.
+version \-Nlastpat points to with the most recently checked in version.
+It then updates \-Nlastpat to point to the current version.
 It leaves the diff sitting in the bugs subdirectory for patmake to pick up.
 It can either use rcsdiff, or a diff command of your choice specified when
 you run packinit, in case your diff is better than rcsdiff.
@@ -408,9 +408,9 @@ may be used from the top level directory or within the \fIbugs\fR directory.
 It will list all the patches and their \fISubject:\fR lines. This program
 knows about compressed patches and will decompress them while producing
 the listing.
-'''
-''' R C S   L a y e r
-'''
+.\"
+.\" R C S   L a y e r
+.\"
 .SH RCS LAYER
 This section describes the RCS layer, in case  something in the tools breaks,
 so that you may fix your RCS files and restart the operation.
-- 
2.0.0.rc0

